<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<HTML>
<HEAD>
<TITLE>untitled</TITLE>
<META http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
<META name="generator" content="pdftohtml 0.35beta">
<META name="date" content="2006-07-28T08:03:46+00:00">
</HEAD>
<BODY bgcolor="#A0A0A0" vlink="blue" link="blue">
<!-- Page 1 -->
<a name="1"></a>
<DIV style="position:relative;width:918;height:1188;">
<STYLE type="text/css">
<!--
	.ft0{font-size:33px;font-family:Times;color:#000000;}
	.ft1{font-size:14px;font-family:Times;color:#000000;}
	.ft2{font-size:9px;font-family:Times;color:#000000;}
	.ft3{font-size:15px;font-family:Times;color:#000000;}
	.ft4{font-size:11px;font-family:Times;color:#000000;}
	.ft5{font-size:37px;font-family:Times;color:#000000;}
	.ft6{font-size:8px;font-family:Times;color:#000000;}
	.ft7{font-size:12px;font-family:Times;color:#000000;}
	.ft8{font-size:6px;font-family:Times;color:#000000;}
	.ft9{font-size:9px;line-height:15px;font-family:Times;color:#000000;}
	.ft10{font-size:12px;line-height:16px;font-family:Times;color:#000000;}
	.ft11{font-size:12px;line-height:19px;font-family:Times;color:#000000;}
	.ft12{font-size:12px;line-height:17px;font-family:Times;color:#000000;}
	.ft13{font-size:9px;line-height:13px;font-family:Times;color:#000000;}
-->
</STYLE>
<IMG width="918" height="1188" src="178001.png" alt="background image">
<DIV style="position:absolute;top:111;left:154"><nobr><span class="ft0">Significance of Gene Ranking for</span></nobr></DIV>
<DIV style="position:absolute;top:153;left:126"><nobr><span class="ft0">Classification of Microarray Samples</span></nobr></DIV>
<DIV style="position:absolute;top:206;left:235"><nobr><span class="ft1">Chaolin Zhang, Xuesong Lu, and Xuegong Zhang</span></nobr></DIV>
<DIV style="position:absolute;top:252;left:75"><nobr><span class="ft3">Abstract</span></nobr></DIV>
<DIV style="position:absolute;top:252;left:75"><nobr><span class="ft9">Many methods for classification and gene selection with microarray data have been developed. These methods usually<br>give a ranking of genes. Evaluating the statistical significance of the gene ranking is important for understanding the results and for<br>further biological investigations, but this question has not been well addressed for machine learning methods in existing works. Here,<br>we address this problem by formulating it in the framework of hypothesis testing and propose a solution based on resampling. The<br>proposed r-test methods convert gene ranking results into position p-values to evaluate the significance of genes. The methods are<br>tested on three real microarray data sets and three simulation data sets with support vector machines as the method of classification<br>and gene selection. The obtained position p-values help to determine the number of genes to be selected and enable scientists to<br>analyze selection results by sophisticated multivariate methods under the same statistical inference paradigm as for simple hypothesis<br>testing methods.</span></nobr></DIV>
<DIV style="position:absolute;top:411;left:75"><nobr><span class="ft2">Keywords</span></nobr></DIV>
<DIV style="position:absolute;top:411;left:75"><nobr><span class="ft2">Significance of gene ranking, gene selection, classification, microarray data analysis.</span></nobr></DIV>
<DIV style="position:absolute;top:435;left:418"><nobr><span class="ft3"></span></nobr></DIV>
<DIV style="position:absolute;top:477;left:39"><nobr><span class="ft3">1</span></nobr></DIV>
<DIV style="position:absolute;top:477;left:66"><nobr><span class="ft3">INTRODUCTION</span></nobr></DIV>
<DIV style="position:absolute;top:502;left:39"><nobr><span class="ft5">A</span></nobr></DIV>
<DIV style="position:absolute;top:505;left:71"><nobr><span class="ft6">N</span></nobr></DIV>
<DIV style="position:absolute;top:503;left:83"><nobr><span class="ft7">important application of DNA microarray technolo-</span></nobr></DIV>
<DIV style="position:absolute;top:520;left:71"><nobr><span class="ft7">gies in functional genomics is to classify samples</span></nobr></DIV>
<DIV style="position:absolute;top:536;left:39"><nobr><span class="ft10">according to their gene expression profiles, e.g., to classify<br>cancer versus normal samples or to classify different types<br>or subtypes of cancer. Selecting genes that are informative<br>for the classification is one key issue for understanding the<br>biology behind the classification and an important step<br>toward discovering those genes responsible for the distinc-<br>tion. For this purpose, researchers have applied a number of<br>test statistics or discriminant criteria to find genes that are<br>differentially expressed between the investigated classes<br>[1], [2], [3], [4], [5], [6], [7]. This category of gene selection<br>methods is usually referred to as the filtering method since<br>the gene selection step usually plays the role of filtering the<br>genes before doing classification with some other methods.<br>Another category of methods is the so-called wrapper<br>methods, which use the classification performance itself as<br>the criterion for selecting the genes and genes are usually<br>selected in a recursive fashion [8], [9], [10], [11], [12]. A<br>representative method of this category is SVM-RFE based<br>on support vector machines (SVM), which uses linear SVM<br>to classify the samples and ranks the contribution of the<br>genes in the classifier by their squared weights [10].</span></nobr></DIV>
<DIV style="position:absolute;top:882;left:57"><nobr><span class="ft7">All these selection methods produce rankings of the</span></nobr></DIV>
<DIV style="position:absolute;top:901;left:39"><nobr><span class="ft11">genes. When a test statistic, such as the t-test, F-test, or<br>bootstrap test, is used as the criterion, the ranking is<br>attached by p-values derived from the null distribution of<br>the test statistic, which reflects the probability of a gene</span></nobr></DIV>
<DIV style="position:absolute;top:502;left:434"><nobr><span class="ft10">showing the observed difference between the classes simply<br>due to chance. Such p-values give biologists a clear<br>understanding of the information that the genes probably<br>contain. The availability of the p-value makes it possible to<br>investigate the microarray data under the solid framework<br>of statistical inference and many theoretical works have<br>been built based on the extension of the concept of p-value,<br>such as the false discovery rate (FDR) study [13].</span></nobr></DIV>
<DIV style="position:absolute;top:638;left:452"><nobr><span class="ft7">Existing gene selection methods that come with p-values</span></nobr></DIV>
<DIV style="position:absolute;top:655;left:434"><nobr><span class="ft12">are of the filtering category and are all univariate methods.<br>To consider possible combinatorial effects of genes, most<br>wrapper methods adopt more sophisticated multivariate<br>machine learning strategies such as SVMs and neural<br>networks. These have been shown in many experiments<br>to be more powerful in terms of classification accuracy.<br>However, for gene selection, the gene rankings produced<br>with these methods do not come with a measure of<br>statistical significance. The ranking is only a relative order<br>of genes according to their relevance to the classifier. There<br>is no clear evaluation of a gene's contribution to the<br>classification. For example, if a gene is ranked 50th<br>according to its weight in the SVM classifier, it is only<br>safe to say that this gene is perhaps more informative<br>than the gene ranked at 51st. However, there is no way to<br>describe how significant it is and there is no ground to<br>compare the information it contains with a gene also<br>ranked as 50th by the same method in another experi-<br>ment. This nature of relative ranking makes it hard to<br>interpret and further explore the gene selection results<br>achieved with such advanced machine learning methods.<br>For example, it is usually difficult to decide on the proper<br>number of genes to be selected in a specific study with<br>such machine learning methods. Most existing works<br>usually select a subgroup of genes with some heuristi-<br>cally decided numbers or thresholds [6], [8], [10]. The<br>advanced estimation techniques, such as FDR, based on<br>significance measures do not apply for such methods.</span></nobr></DIV>
<DIV style="position:absolute;top:70;left:38"><nobr><span class="ft6">312</span></nobr></DIV>
<DIV style="position:absolute;top:70;left:169"><nobr><span class="ft6">IEEE/ACM TRANSACTIONS ON COMPUTATIONAL BIOLOGY AND BIOINFORMATICS,</span></nobr></DIV>
<DIV style="position:absolute;top:70;left:601"><nobr><span class="ft6">VOL. 3,</span></nobr></DIV>
<DIV style="position:absolute;top:70;left:648"><nobr><span class="ft6">NO. 3,</span></nobr></DIV>
<DIV style="position:absolute;top:70;left:689"><nobr><span class="ft6">JULY-SEPTEMBER 2006</span></nobr></DIV>
<DIV style="position:absolute;top:1008;left:39"><nobr><span class="ft2">.</span></nobr></DIV>
<DIV style="position:absolute;top:1007;left:54"><nobr><span class="ft13">C. Zhang is with the Cold Spring Harbor Laboratory and the Department<br>of Biomedical Engineering, State University of New York at Stony Brook,<br>NY 11794. E-mail: zhangc@cshl.edu.</span></nobr></DIV>
<DIV style="position:absolute;top:1048;left:39"><nobr><span class="ft2">.</span></nobr></DIV>
<DIV style="position:absolute;top:1048;left:54"><nobr><span class="ft13">X. Lu and X. Zhang are with the MOE Key Laboratory of Bioinformatics/<br>Department of Automation, Tsinghua University, Beijing 100084, China.<br>E-mail: lxs97@mails.tsinghua.edu.cn, zhangxg@tsinghua.edu.cn.</span></nobr></DIV>
<DIV style="position:absolute;top:1094;left:39"><nobr><span class="ft13">Manuscript received 28 Sept. 2004; revised 14 May 2005; accepted 5 Oct.<br>2005; published online 31 July 2006.<br>For information on obtaining reprints of this article, please send e-mail to:<br>tcbb@computer.org, and reference IEEECS Log Number TCBB-0159-0904.</span></nobr></DIV>
<DIV style="position:absolute;top:1159;left:272"><nobr><span class="ft8">1545-5963/06/$20.00 ß 2006 IEEE</span></nobr></DIV>
<DIV style="position:absolute;top:1159;left:434"><nobr><span class="ft8">Published by the IEEE CS, CI, and EMB Societies &amp; the ACM</span></nobr></DIV>
</DIV>
<!-- Page 2 -->
<a name="2"></a>
<DIV style="position:relative;width:918;height:1188;">
<STYLE type="text/css">
<!--
	.ft14{font-size:7px;font-family:Times;color:#000000;}
-->
</STYLE>
<IMG width="918" height="1188" src="178002.png" alt="background image">
<DIV style="position:absolute;top:102;left:57"><nobr><span class="ft7">Evaluating the statistical significance of the detected</span></nobr></DIV>
<DIV style="position:absolute;top:120;left:39"><nobr><span class="ft12">signal is the central idea in the paradigm of statistical<br>inference from experimental data. There should be an<br>equivalent study on those machine-learning-based multi-<br>variate gene selection methods which produce ranks<br>according to their own criteria. Strategies such as permuta-<br>tion can be utilized to assess the significance of the<br>classification accuracy, but they do not measure the<br>significance of the selected genes directly. Surprisingly, this<br>question has not been addressed by the statistics or<br>bioinformatics community in existing literature. We there-<br>fore propose that the question be asked in this way: For an<br>observed ranking of genes by a certain method, what is the<br>probability that a gene is ranked at or above the observed<br>position due to chance (by the same method) if the gene is,<br>in fact, not informative to the classification? ("Being<br>informative" is in the sense of the criteria defined or<br>implied by the classification and ranking method. It may<br>have different meanings for different methods.) We call this<br>problem the significance of gene ranking or feature ranking.<br>We raise this problem in this paper and describe our<br>strategy toward a solution. The problem is discussed in the<br>context of microarray classification of cancer samples, but<br>the philosophy and methodology is not restricted to this<br>scenario.</span></nobr></DIV>
<DIV style="position:absolute;top:569;left:39"><nobr><span class="ft3">2</span></nobr></DIV>
<DIV style="position:absolute;top:569;left:66"><nobr><span class="ft3">THE SIGNIFICANCE OF RANKING PROBLEM</span></nobr></DIV>
<DIV style="position:absolute;top:594;left:39"><nobr><span class="ft7">Suppose a</span></nobr></DIV>
<DIV style="position:absolute;top:594;left:130"><nobr><span class="ft7">microarray</span></nobr></DIV>
<DIV style="position:absolute;top:594;left:215"><nobr><span class="ft7">data</span></nobr></DIV>
<DIV style="position:absolute;top:594;left:257"><nobr><span class="ft7">set</span></nobr></DIV>
<DIV style="position:absolute;top:594;left:289"><nobr><span class="ft7">contains</span></nobr></DIV>
<DIV style="position:absolute;top:595;left:356"><nobr><span class="ft7">m</span></nobr></DIV>
<DIV style="position:absolute;top:594;left:383"><nobr><span class="ft7">cases</span></nobr></DIV>
<DIV style="position:absolute;top:613;left:39"><nobr><span class="ft7">X</span></nobr></DIV>
<DIV style="position:absolute;top:612;left:55"><nobr><span class="ft7">¼ fx</span></nobr></DIV>
<DIV style="position:absolute;top:618;left:86"><nobr><span class="ft14">i</span></nobr></DIV>
<DIV style="position:absolute;top:613;left:90"><nobr><span class="ft7">; i</span></nobr></DIV>
<DIV style="position:absolute;top:612;left:105"><nobr><span class="ft7">¼ 1;</span></nobr></DIV>
<DIV style="position:absolute;top:613;left:152"><nobr><span class="ft7">; m</span></nobr></DIV>
<DIV style="position:absolute;top:612;left:171"><nobr><span class="ft7">g. Each case is characterized by a</span></nobr></DIV>
<DIV style="position:absolute;top:630;left:39"><nobr><span class="ft7">vector</span></nobr></DIV>
<DIV style="position:absolute;top:630;left:100"><nobr><span class="ft7">of</span></nobr></DIV>
<DIV style="position:absolute;top:630;left:129"><nobr><span class="ft7">the</span></nobr></DIV>
<DIV style="position:absolute;top:630;left:167"><nobr><span class="ft7">expression</span></nobr></DIV>
<DIV style="position:absolute;top:630;left:261"><nobr><span class="ft7">values</span></nobr></DIV>
<DIV style="position:absolute;top:630;left:323"><nobr><span class="ft7">of</span></nobr></DIV>
<DIV style="position:absolute;top:630;left:352"><nobr><span class="ft7">n</span></nobr></DIV>
<DIV style="position:absolute;top:630;left:376"><nobr><span class="ft7">genes</span></nobr></DIV>
<DIV style="position:absolute;top:648;left:39"><nobr><span class="ft7">x</span></nobr></DIV>
<DIV style="position:absolute;top:653;left:48"><nobr><span class="ft14">i</span></nobr></DIV>
<DIV style="position:absolute;top:647;left:56"><nobr><span class="ft7">¼ ½x</span></nobr></DIV>
<DIV style="position:absolute;top:653;left:83"><nobr><span class="ft14">i1</span></nobr></DIV>
<DIV style="position:absolute;top:648;left:92"><nobr><span class="ft7">; x</span></nobr></DIV>
<DIV style="position:absolute;top:653;left:106"><nobr><span class="ft14">i2</span></nobr></DIV>
<DIV style="position:absolute;top:648;left:115"><nobr><span class="ft7">;</span></nobr></DIV>
<DIV style="position:absolute;top:648;left:140"><nobr><span class="ft7">; x</span></nobr></DIV>
<DIV style="position:absolute;top:653;left:154"><nobr><span class="ft14">in T</span></nobr></DIV>
<DIV style="position:absolute;top:647;left:180"><nobr><span class="ft7">2 R</span></nobr></DIV>
<DIV style="position:absolute;top:646;left:204"><nobr><span class="ft14">n</span></nobr></DIV>
<DIV style="position:absolute;top:648;left:210"><nobr><span class="ft7">; i</span></nobr></DIV>
<DIV style="position:absolute;top:647;left:225"><nobr><span class="ft7">¼ 1;</span></nobr></DIV>
<DIV style="position:absolute;top:648;left:272"><nobr><span class="ft7">; m</span></nobr></DIV>
<DIV style="position:absolute;top:648;left:291"><nobr><span class="ft7">. Each gene is a</span></nobr></DIV>
<DIV style="position:absolute;top:665;left:39"><nobr><span class="ft7">vector of their expression values across the cases g</span></nobr></DIV>
<DIV style="position:absolute;top:672;left:396"><nobr><span class="ft14">j</span></nobr></DIV>
<DIV style="position:absolute;top:665;left:405"><nobr><span class="ft7">¼</span></nobr></DIV>
<DIV style="position:absolute;top:683;left:39"><nobr><span class="ft7">½x</span></nobr></DIV>
<DIV style="position:absolute;top:689;left:51"><nobr><span class="ft14">1j</span></nobr></DIV>
<DIV style="position:absolute;top:683;left:61"><nobr><span class="ft7">; x</span></nobr></DIV>
<DIV style="position:absolute;top:689;left:75"><nobr><span class="ft14">2j</span></nobr></DIV>
<DIV style="position:absolute;top:683;left:85"><nobr><span class="ft7">;</span></nobr></DIV>
<DIV style="position:absolute;top:683;left:110"><nobr><span class="ft7">; x</span></nobr></DIV>
<DIV style="position:absolute;top:689;left:124"><nobr><span class="ft14">mj T</span></nobr></DIV>
<DIV style="position:absolute;top:683;left:159"><nobr><span class="ft7">and we denote the set of all genes</span></nobr></DIV>
<DIV style="position:absolute;top:701;left:39"><nobr><span class="ft7">a s</span></nobr></DIV>
<DIV style="position:absolute;top:701;left:72"><nobr><span class="ft7">G</span></nobr></DIV>
<DIV style="position:absolute;top:700;left:87"><nobr><span class="ft7">¼ fg</span></nobr></DIV>
<DIV style="position:absolute;top:707;left:117"><nobr><span class="ft14">j</span></nobr></DIV>
<DIV style="position:absolute;top:701;left:122"><nobr><span class="ft7">; j</span></nobr></DIV>
<DIV style="position:absolute;top:700;left:142"><nobr><span class="ft7">¼ 1;</span></nobr></DIV>
<DIV style="position:absolute;top:701;left:189"><nobr><span class="ft7">; n</span></nobr></DIV>
<DIV style="position:absolute;top:700;left:203"><nobr><span class="ft7">g. E a c h c a s e h a s a l a b e l</span></nobr></DIV>
<DIV style="position:absolute;top:719;left:39"><nobr><span class="ft7">y</span></nobr></DIV>
<DIV style="position:absolute;top:724;left:46"><nobr><span class="ft14">i</span></nobr></DIV>
<DIV style="position:absolute;top:718;left:54"><nobr><span class="ft7">¼ f 1; 1g, i ¼ 1;</span></nobr></DIV>
<DIV style="position:absolute;top:719;left:180"><nobr><span class="ft7">; m</span></nobr></DIV>
<DIV style="position:absolute;top:718;left:204"><nobr><span class="ft7">indicating the class it belongs to</span></nobr></DIV>
<DIV style="position:absolute;top:736;left:39"><nobr><span class="ft12">among the studied two classes, e.g., normal versus cancer,<br>or two subtypes of a cancer, etc. Among the n genes, usually<br>some are informative to the classification and some are not,<br>but we do not know which genes are informative and which<br>are not. For the convenience of description, we denote the<br>set of informative genes as I</span></nobr></DIV>
<DIV style="position:absolute;top:830;left:217"><nobr><span class="ft14">G</span></nobr></DIV>
<DIV style="position:absolute;top:825;left:229"><nobr><span class="ft7">and that of the uninformative</span></nobr></DIV>
<DIV style="position:absolute;top:842;left:39"><nobr><span class="ft7">genes as U</span></nobr></DIV>
<DIV style="position:absolute;top:848;left:107"><nobr><span class="ft14">G</span></nobr></DIV>
<DIV style="position:absolute;top:842;left:116"><nobr><span class="ft7">. To simplify the problem, we assume that</span></nobr></DIV>
<DIV style="position:absolute;top:872;left:132"><nobr><span class="ft7">I</span></nobr></DIV>
<DIV style="position:absolute;top:877;left:139"><nobr><span class="ft14">G</span></nobr></DIV>
<DIV style="position:absolute;top:871;left:150"><nobr><span class="ft7">\ U</span></nobr></DIV>
<DIV style="position:absolute;top:877;left:172"><nobr><span class="ft14">G</span></nobr></DIV>
<DIV style="position:absolute;top:871;left:185"><nobr><span class="ft7">¼</span></nobr></DIV>
<DIV style="position:absolute;top:871;left:214"><nobr><span class="ft7">and I</span></nobr></DIV>
<DIV style="position:absolute;top:877;left:247"><nobr><span class="ft14">G</span></nobr></DIV>
<DIV style="position:absolute;top:871;left:258"><nobr><span class="ft7">[ U</span></nobr></DIV>
<DIV style="position:absolute;top:877;left:281"><nobr><span class="ft14">G</span></nobr></DIV>
<DIV style="position:absolute;top:871;left:293"><nobr><span class="ft7">¼ G:</span></nobr></DIV>
<DIV style="position:absolute;top:871;left:398"><nobr><span class="ft7">ð1Þ</span></nobr></DIV>
<DIV style="position:absolute;top:898;left:39"><nobr><span class="ft7">The goal is to build a classifier that can predict the classes ^</span></nobr></DIV>
<DIV style="position:absolute;top:898;left:405"><nobr><span class="ft7">y</span></nobr></DIV>
<DIV style="position:absolute;top:903;left:412"><nobr><span class="ft14">i</span></nobr></DIV>
<DIV style="position:absolute;top:916;left:39"><nobr><span class="ft7">of the cases from x</span></nobr></DIV>
<DIV style="position:absolute;top:921;left:164"><nobr><span class="ft14">i</span></nobr></DIV>
<DIV style="position:absolute;top:916;left:174"><nobr><span class="ft7">and, at the same time, to identify the</span></nobr></DIV>
<DIV style="position:absolute;top:933;left:39"><nobr><span class="ft7">genes that most likely belong to I</span></nobr></DIV>
<DIV style="position:absolute;top:939;left:247"><nobr><span class="ft14">G</span></nobr></DIV>
<DIV style="position:absolute;top:933;left:255"><nobr><span class="ft7">. The former task is called</span></nobr></DIV>
<DIV style="position:absolute;top:951;left:39"><nobr><span class="ft12">classification and the latter one is called gene selection. In<br>the current study, we assume that there has already been a<br>ranking method RM which produces a ranking position for<br>each gene according to some criterion assessing the gene's<br>relevance with the classification:</span></nobr></DIV>
<DIV style="position:absolute;top:1052;left:77"><nobr><span class="ft7">r</span></nobr></DIV>
<DIV style="position:absolute;top:1057;left:84"><nobr><span class="ft14">j</span></nobr></DIV>
<DIV style="position:absolute;top:1051;left:93"><nobr><span class="ft7">¼ rank g</span></nobr></DIV>
<DIV style="position:absolute;top:1057;left:150"><nobr><span class="ft14">j</span></nobr></DIV>
<DIV style="position:absolute;top:1051;left:155"><nobr><span class="ft7">j ðx</span></nobr></DIV>
<DIV style="position:absolute;top:1057;left:180"><nobr><span class="ft14">i</span></nobr></DIV>
<DIV style="position:absolute;top:1052;left:184"><nobr><span class="ft7">; y</span></nobr></DIV>
<DIV style="position:absolute;top:1057;left:197"><nobr><span class="ft14">i</span></nobr></DIV>
<DIV style="position:absolute;top:1051;left:201"><nobr><span class="ft7">Þ; i ¼ 1;</span></nobr></DIV>
<DIV style="position:absolute;top:1052;left:268"><nobr><span class="ft7">; m</span></nobr></DIV>
<DIV style="position:absolute;top:1051;left:158"><nobr><span class="ft7">f</span></nobr></DIV>
<DIV style="position:absolute;top:1051;left:287"><nobr><span class="ft7">g ; j ¼ 1;</span></nobr></DIV>
<DIV style="position:absolute;top:1052;left:363"><nobr><span class="ft7">; n</span></nobr></DIV>
<DIV style="position:absolute;top:1051;left:398"><nobr><span class="ft7">ð2Þ</span></nobr></DIV>
<DIV style="position:absolute;top:1080;left:39"><nobr><span class="ft12">and we do not distinguish the specific types of the RM. The<br>ranking is obtained based on the samples, thus r</span></nobr></DIV>
<DIV style="position:absolute;top:1103;left:378"><nobr><span class="ft14">j</span></nobr></DIV>
<DIV style="position:absolute;top:1098;left:391"><nobr><span class="ft7">is a</span></nobr></DIV>
<DIV style="position:absolute;top:1115;left:39"><nobr><span class="ft12">random variable. The significance-of-ranking problem is to<br>calculate the following probability:</span></nobr></DIV>
<DIV style="position:absolute;top:102;left:517"><nobr><span class="ft7">p</span></nobr></DIV>
<DIV style="position:absolute;top:101;left:524"><nobr><span class="ft7">ðr</span></nobr></DIV>
<DIV style="position:absolute;top:107;left:536"><nobr><span class="ft14">j</span></nobr></DIV>
<DIV style="position:absolute;top:101;left:541"><nobr><span class="ft7">Þ ¼</span></nobr></DIV>
<DIV style="position:absolute;top:96;left:551"><nobr><span class="ft14">4</span></nobr></DIV>
<DIV style="position:absolute;top:102;left:565"><nobr><span class="ft7">P rank</span></nobr></DIV>
<DIV style="position:absolute;top:101;left:612"><nobr><span class="ft7">ðg</span></nobr></DIV>
<DIV style="position:absolute;top:107;left:624"><nobr><span class="ft14">j</span></nobr></DIV>
<DIV style="position:absolute;top:101;left:629"><nobr><span class="ft7">Þ</span></nobr></DIV>
<DIV style="position:absolute;top:102;left:653"><nobr><span class="ft7">r</span></nobr></DIV>
<DIV style="position:absolute;top:107;left:659"><nobr><span class="ft14">j</span></nobr></DIV>
<DIV style="position:absolute;top:101;left:664"><nobr><span class="ft7">jg</span></nobr></DIV>
<DIV style="position:absolute;top:107;left:674"><nobr><span class="ft14">j</span></nobr></DIV>
<DIV style="position:absolute;top:101;left:683"><nobr><span class="ft7">2 U</span></nobr></DIV>
<DIV style="position:absolute;top:107;left:706"><nobr><span class="ft14">G</span></nobr></DIV>
<DIV style="position:absolute;top:102;left:721"><nobr><span class="ft7">;</span></nobr></DIV>
<DIV style="position:absolute;top:101;left:792"><nobr><span class="ft7">ð3Þ</span></nobr></DIV>
<DIV style="position:absolute;top:129;left:434"><nobr><span class="ft12">i.e., given a gene is uninformative to the classification<br>(according to RM's criterion), what is the probability that it<br>is ranked at or above the observed ranking position by the<br>ranking method? We call this probability the p-value of a<br>gene's ranking position or, simply, position p-value.</span></nobr></DIV>
<DIV style="position:absolute;top:216;left:452"><nobr><span class="ft7">This significance-of-ranking problem is distinct from</span></nobr></DIV>
<DIV style="position:absolute;top:233;left:434"><nobr><span class="ft12">existing statistics for testing differentially expressed genes<br>in several aspects. It applies to more complicated multi-<br>variate classification and gene selection methods. Even<br>when it is applied on gene ranking methods based on<br>univariate hypothesis tests like t-test, the position p-value is<br>different with the t-test p-value by definition. The t-test<br>p-value of a gene is calculated from the expression values of<br>this gene in the two sample sets by comparing with the<br>assumed null distribution model when the gene is not<br>differentially expressed in the two classes. The position<br>p-value of a single gene, however, is defined on its context,<br>in the sense that its value depends not only on the<br>expression of this gene in the samples, but also on other<br>genes in the same data set. A gene with the same expression<br>values may have different position p-values in different<br>data sets. The null distributions of ranks of uninformative<br>genes are different in different data sets and, therefore, the<br>foremost challenge for solving the problem is that the null<br>distribution has to be estimated from the specific data set<br>under investigation.</span></nobr></DIV>
<DIV style="position:absolute;top:603;left:434"><nobr><span class="ft3">3</span></nobr></DIV>
<DIV style="position:absolute;top:603;left:461"><nobr><span class="ft3">THE R-TEST SCHEME</span></nobr></DIV>
<DIV style="position:absolute;top:628;left:434"><nobr><span class="ft12">The significance-of-ranking problem is formulated as a<br>hypothesis testing problem. The null hypothesis is that the<br>gene is not informative or g</span></nobr></DIV>
<DIV style="position:absolute;top:668;left:614"><nobr><span class="ft14">j</span></nobr></DIV>
<DIV style="position:absolute;top:663;left:623"><nobr><span class="ft7">2 U</span></nobr></DIV>
<DIV style="position:absolute;top:668;left:646"><nobr><span class="ft14">G</span></nobr></DIV>
<DIV style="position:absolute;top:663;left:654"><nobr><span class="ft7">, the alternative hypoth-</span></nobr></DIV>
<DIV style="position:absolute;top:680;left:434"><nobr><span class="ft7">esis is g</span></nobr></DIV>
<DIV style="position:absolute;top:686;left:491"><nobr><span class="ft14">j</span></nobr></DIV>
<DIV style="position:absolute;top:680;left:500"><nobr><span class="ft7">2 I</span></nobr></DIV>
<DIV style="position:absolute;top:686;left:519"><nobr><span class="ft14">G</span></nobr></DIV>
<DIV style="position:absolute;top:680;left:536"><nobr><span class="ft7">(the gene is informative) since we have</span></nobr></DIV>
<DIV style="position:absolute;top:697;left:434"><nobr><span class="ft12">assumption (1) and the statistic to be used to test the<br>hypothesis is the ranking position. As in standard hypoth-<br>esis testing, the key to solving the problem is to obtain the<br>distribution of the statistic under the null hypothesis, i.e.,<br>the distribution of the ranks of uninformative genes:</span></nobr></DIV>
<DIV style="position:absolute;top:796;left:583"><nobr><span class="ft7">P r</span></nobr></DIV>
<DIV style="position:absolute;top:795;left:606"><nobr><span class="ft7">jg 2 U</span></nobr></DIV>
<DIV style="position:absolute;top:801;left:643"><nobr><span class="ft14">G</span></nobr></DIV>
<DIV style="position:absolute;top:795;left:594"><nobr><span class="ft7">ð</span></nobr></DIV>
<DIV style="position:absolute;top:795;left:652"><nobr><span class="ft7">Þ:</span></nobr></DIV>
<DIV style="position:absolute;top:795;left:792"><nobr><span class="ft7">ð4Þ</span></nobr></DIV>
<DIV style="position:absolute;top:822;left:434"><nobr><span class="ft7">For the extreme case when I</span></nobr></DIV>
<DIV style="position:absolute;top:827;left:640"><nobr><span class="ft14">G</span></nobr></DIV>
<DIV style="position:absolute;top:822;left:653"><nobr><span class="ft7">¼</span></nobr></DIV>
<DIV style="position:absolute;top:822;left:687"><nobr><span class="ft7">(all the genes are</span></nobr></DIV>
<DIV style="position:absolute;top:839;left:434"><nobr><span class="ft12">uninformative) and the ranking method is not biased, it is<br>obvious that the null distribution is uniform. In a real<br>microarray data set, however, usually some genes are<br>informative and some are not, thus the uniform null<br>distribution is not applicable. The null rank distribution in<br>a practical investigation depends on many factors, includ-<br>ing the separability of the two classes, the underlying<br>number of informative genes, the power of the ranking<br>method, the sample size, etc. The characteristics of these<br>factors are not well understood in either statistics or biology<br>and, therefore, we have to estimate an empirical null<br>distribution from the data set itself.</span></nobr></DIV>
<DIV style="position:absolute;top:1046;left:452"><nobr><span class="ft7">We propose to tackle this problem in two steps. First, we</span></nobr></DIV>
<DIV style="position:absolute;top:1064;left:434"><nobr><span class="ft12">identify a set of putative uninformative genes (PUGs) which<br>are a subset of U</span></nobr></DIV>
<DIV style="position:absolute;top:1087;left:551"><nobr><span class="ft14">G</span></nobr></DIV>
<DIV style="position:absolute;top:1081;left:560"><nobr><span class="ft7">. This is possible in practice because,</span></nobr></DIV>
<DIV style="position:absolute;top:1098;left:434"><nobr><span class="ft7">although we do not know U</span></nobr></DIV>
<DIV style="position:absolute;top:1104;left:605"><nobr><span class="ft14">G</span></nobr></DIV>
<DIV style="position:absolute;top:1098;left:614"><nobr><span class="ft7">, discovering a number of genes</span></nobr></DIV>
<DIV style="position:absolute;top:1116;left:434"><nobr><span class="ft12">that are irrelevant to the classification is usually not hard in<br>most microarray data sets. We denote the identified subset as</span></nobr></DIV>
<DIV style="position:absolute;top:70;left:38"><nobr><span class="ft6">ZHANG ET AL.: SIGNIFICANCE OF GENE RANKING FOR CLASSIFICATION OF MICROARRAY SAMPLES</span></nobr></DIV>
<DIV style="position:absolute;top:70;left:792"><nobr><span class="ft6">313</span></nobr></DIV>
</DIV>
<!-- Page 3 -->
<a name="3"></a>
<DIV style="position:relative;width:918;height:1188;">
<STYLE type="text/css">
<!--
	.ft15{font-size:7px;line-height:9px;font-family:Times;color:#000000;}
	.ft16{font-size:12px;line-height:18px;font-family:Times;color:#000000;}
	.ft17{font-size:9px;line-height:14px;font-family:Times;color:#000000;}
-->
</STYLE>
<IMG width="918" height="1188" src="178003.png" alt="background image">
<DIV style="position:absolute;top:693;left:39"><nobr><span class="ft7">U</span></nobr></DIV>
<DIV style="position:absolute;top:691;left:50"><nobr><span class="ft14">0G</span></nobr></DIV>
<DIV style="position:absolute;top:693;left:57"><nobr><span class="ft7">. The next step is to estimate the null distribution of ranks</span></nobr></DIV>
<DIV style="position:absolute;top:711;left:39"><nobr><span class="ft7">with the ranking positions of these PUGs.</span></nobr></DIV>
<DIV style="position:absolute;top:728;left:57"><nobr><span class="ft7">From the original data set, we resample L new data sets</span></nobr></DIV>
<DIV style="position:absolute;top:746;left:39"><nobr><span class="ft12">and apply the ranking method on each of them, producing,<br>for each gene L, ranks r</span></nobr></DIV>
<DIV style="position:absolute;top:762;left:198"><nobr><span class="ft15">l<br>j</span></nobr></DIV>
<DIV style="position:absolute;top:764;left:203"><nobr><span class="ft7">; l</span></nobr></DIV>
<DIV style="position:absolute;top:763;left:218"><nobr><span class="ft7">¼ 1;</span></nobr></DIV>
<DIV style="position:absolute;top:764;left:264"><nobr><span class="ft7">; L</span></nobr></DIV>
<DIV style="position:absolute;top:764;left:280"><nobr><span class="ft7">. In our implementa-</span></nobr></DIV>
<DIV style="position:absolute;top:782;left:39"><nobr><span class="ft12">tion, we randomly resample half of the cases in the original<br>data set each time. Other resampling schemes such as<br>bootstrapping can also be used to obtain similar results<br>according to our experiments (data not shown). Since,<br>usually, the size of U</span></nobr></DIV>
<DIV style="position:absolute;top:859;left:176"><nobr><span class="ft14">G</span></nobr></DIV>
<DIV style="position:absolute;top:853;left:190"><nobr><span class="ft7">is much larger than that of I</span></nobr></DIV>
<DIV style="position:absolute;top:859;left:376"><nobr><span class="ft14">G</span></nobr></DIV>
<DIV style="position:absolute;top:853;left:390"><nobr><span class="ft7">(i.e.,</span></nobr></DIV>
<DIV style="position:absolute;top:871;left:39"><nobr><span class="ft12">most genes are uninformative), if a gene tends to always be<br>ranked at the bottom in the L rankings, it is very likely that<br>the gene is an uninformative one. Thus, we define r</span></nobr></DIV>
<DIV style="position:absolute;top:913;left:370"><nobr><span class="ft14">j</span></nobr></DIV>
<DIV style="position:absolute;top:907;left:379"><nobr><span class="ft7">as the</span></nobr></DIV>
<DIV style="position:absolute;top:925;left:39"><nobr><span class="ft7">average position of gene j in the L rankings,</span></nobr></DIV>
<DIV style="position:absolute;top:968;left:147"><nobr><span class="ft7">r</span></nobr></DIV>
<DIV style="position:absolute;top:973;left:153"><nobr><span class="ft14">j</span></nobr></DIV>
<DIV style="position:absolute;top:967;left:162"><nobr><span class="ft7">¼ 1</span></nobr></DIV>
<DIV style="position:absolute;top:978;left:177"><nobr><span class="ft7">L</span></nobr></DIV>
<DIV style="position:absolute;top:953;left:189"><nobr><span class="ft7">X</span></nobr></DIV>
<DIV style="position:absolute;top:954;left:196"><nobr><span class="ft14">L</span></nobr></DIV>
<DIV style="position:absolute;top:986;left:191"><nobr><span class="ft14">l</span></nobr></DIV>
<DIV style="position:absolute;top:986;left:194"><nobr><span class="ft14">¼1</span></nobr></DIV>
<DIV style="position:absolute;top:968;left:212"><nobr><span class="ft7">r</span></nobr></DIV>
<DIV style="position:absolute;top:965;left:218"><nobr><span class="ft15">l<br>j</span></nobr></DIV>
<DIV style="position:absolute;top:968;left:223"><nobr><span class="ft7">; j</span></nobr></DIV>
<DIV style="position:absolute;top:967;left:247"><nobr><span class="ft7">¼ 1;</span></nobr></DIV>
<DIV style="position:absolute;top:968;left:293"><nobr><span class="ft7">; n</span></nobr></DIV>
<DIV style="position:absolute;top:967;left:398"><nobr><span class="ft7">ð5Þ</span></nobr></DIV>
<DIV style="position:absolute;top:1008;left:39"><nobr><span class="ft7">and select the bottom k genes with the largest r</span></nobr></DIV>
<DIV style="position:absolute;top:1013;left:365"><nobr><span class="ft14">j</span></nobr></DIV>
<DIV style="position:absolute;top:1008;left:376"><nobr><span class="ft7">as the</span></nobr></DIV>
<DIV style="position:absolute;top:1025;left:39"><nobr><span class="ft7">PUGs to form U</span></nobr></DIV>
<DIV style="position:absolute;top:1023;left:144"><nobr><span class="ft14">0G</span></nobr></DIV>
<DIV style="position:absolute;top:1025;left:151"><nobr><span class="ft7">, where k is a preset number. We rewrite</span></nobr></DIV>
<DIV style="position:absolute;top:1044;left:39"><nobr><span class="ft7">U</span></nobr></DIV>
<DIV style="position:absolute;top:1041;left:50"><nobr><span class="ft14">0G</span></nobr></DIV>
<DIV style="position:absolute;top:1043;left:62"><nobr><span class="ft7">as U</span></nobr></DIV>
<DIV style="position:absolute;top:1041;left:91"><nobr><span class="ft14">0k</span></nobr></DIV>
<DIV style="position:absolute;top:1051;left:90"><nobr><span class="ft14">G</span></nobr></DIV>
<DIV style="position:absolute;top:1043;left:104"><nobr><span class="ft7">when we need to emphasize the role of k in this</span></nobr></DIV>
<DIV style="position:absolute;top:1061;left:39"><nobr><span class="ft7">procedure. We assume that U</span></nobr></DIV>
<DIV style="position:absolute;top:1059;left:232"><nobr><span class="ft14">0k</span></nobr></DIV>
<DIV style="position:absolute;top:1069;left:231"><nobr><span class="ft14">G</span></nobr></DIV>
<DIV style="position:absolute;top:1061;left:246"><nobr><span class="ft7">is a random sample of U</span></nobr></DIV>
<DIV style="position:absolute;top:1067;left:407"><nobr><span class="ft14">G</span></nobr></DIV>
<DIV style="position:absolute;top:1079;left:39"><nobr><span class="ft7">and r</span></nobr></DIV>
<DIV style="position:absolute;top:1077;left:74"><nobr><span class="ft15">l<br>j</span></nobr></DIV>
<DIV style="position:absolute;top:1079;left:78"><nobr><span class="ft7">, l ¼ 1;</span></nobr></DIV>
<DIV style="position:absolute;top:1080;left:141"><nobr><span class="ft7">; L</span></nobr></DIV>
<DIV style="position:absolute;top:1079;left:160"><nobr><span class="ft7">for g</span></nobr></DIV>
<DIV style="position:absolute;top:1085;left:189"><nobr><span class="ft14">j</span></nobr></DIV>
<DIV style="position:absolute;top:1079;left:198"><nobr><span class="ft7">2 U</span></nobr></DIV>
<DIV style="position:absolute;top:1077;left:222"><nobr><span class="ft14">0k</span></nobr></DIV>
<DIV style="position:absolute;top:1087;left:220"><nobr><span class="ft14">G</span></nobr></DIV>
<DIV style="position:absolute;top:1079;left:234"><nobr><span class="ft7">is a random sample from the</span></nobr></DIV>
<DIV style="position:absolute;top:1097;left:39"><nobr><span class="ft12">underlying null distribution of the ranks of uninformative<br>genes. Thus, we have k L observations of the null<br>distribution of ranks from which we estimate the null</span></nobr></DIV>
<DIV style="position:absolute;top:693;left:434"><nobr><span class="ft12">distribution using a histogram. More sophisticated non-<br>parametric methods can be adopted to fit the distribution if<br>necessary. We denote the estimated null distribution as</span></nobr></DIV>
<DIV style="position:absolute;top:757;left:513"><nobr><span class="ft7">^</span></nobr></DIV>
<DIV style="position:absolute;top:760;left:510"><nobr><span class="ft7">P r</span></nobr></DIV>
<DIV style="position:absolute;top:760;left:532"><nobr><span class="ft7">jg 2 U</span></nobr></DIV>
<DIV style="position:absolute;top:766;left:570"><nobr><span class="ft14">G</span></nobr></DIV>
<DIV style="position:absolute;top:760;left:520"><nobr><span class="ft7">ð</span></nobr></DIV>
<DIV style="position:absolute;top:760;left:578"><nobr><span class="ft7">Þ ¼ P</span></nobr></DIV>
<DIV style="position:absolute;top:766;left:611"><nobr><span class="ft14">histogram</span></nobr></DIV>
<DIV style="position:absolute;top:760;left:657"><nobr><span class="ft7">ðr</span></nobr></DIV>
<DIV style="position:absolute;top:758;left:669"><nobr><span class="ft15">l<br>j</span></nobr></DIV>
<DIV style="position:absolute;top:760;left:674"><nobr><span class="ft7">jg</span></nobr></DIV>
<DIV style="position:absolute;top:766;left:684"><nobr><span class="ft14">j</span></nobr></DIV>
<DIV style="position:absolute;top:760;left:693"><nobr><span class="ft7">2 U</span></nobr></DIV>
<DIV style="position:absolute;top:757;left:717"><nobr><span class="ft14">0k</span></nobr></DIV>
<DIV style="position:absolute;top:767;left:716"><nobr><span class="ft14">G</span></nobr></DIV>
<DIV style="position:absolute;top:760;left:725"><nobr><span class="ft7">Þ:</span></nobr></DIV>
<DIV style="position:absolute;top:760;left:792"><nobr><span class="ft7">ð6Þ</span></nobr></DIV>
<DIV style="position:absolute;top:789;left:452"><nobr><span class="ft7">With this estimated null distribution, the calculation of</span></nobr></DIV>
<DIV style="position:absolute;top:807;left:434"><nobr><span class="ft12">the position p-value is straightforward: For gene i with<br>ranking position r</span></nobr></DIV>
<DIV style="position:absolute;top:831;left:550"><nobr><span class="ft14">i</span></nobr></DIV>
<DIV style="position:absolute;top:825;left:554"><nobr><span class="ft7">,</span></nobr></DIV>
<DIV style="position:absolute;top:857;left:448"><nobr><span class="ft7">^</span></nobr></DIV>
<DIV style="position:absolute;top:857;left:448"><nobr><span class="ft7">p</span></nobr></DIV>
<DIV style="position:absolute;top:856;left:455"><nobr><span class="ft7">ðr</span></nobr></DIV>
<DIV style="position:absolute;top:862;left:467"><nobr><span class="ft14">i</span></nobr></DIV>
<DIV style="position:absolute;top:856;left:471"><nobr><span class="ft7">Þ ¼ ^</span></nobr></DIV>
<DIV style="position:absolute;top:857;left:495"><nobr><span class="ft7">P</span></nobr></DIV>
<DIV style="position:absolute;top:856;left:506"><nobr><span class="ft7">ðr</span></nobr></DIV>
<DIV style="position:absolute;top:857;left:536"><nobr><span class="ft7">r</span></nobr></DIV>
<DIV style="position:absolute;top:862;left:543"><nobr><span class="ft14">i</span></nobr></DIV>
<DIV style="position:absolute;top:856;left:547"><nobr><span class="ft7">jg 2 U</span></nobr></DIV>
<DIV style="position:absolute;top:862;left:584"><nobr><span class="ft14">G</span></nobr></DIV>
<DIV style="position:absolute;top:856;left:593"><nobr><span class="ft7">Þ ¼ P</span></nobr></DIV>
<DIV style="position:absolute;top:862;left:626"><nobr><span class="ft14">histogram</span></nobr></DIV>
<DIV style="position:absolute;top:856;left:672"><nobr><span class="ft7">ðr</span></nobr></DIV>
<DIV style="position:absolute;top:854;left:683"><nobr><span class="ft15">l<br>j</span></nobr></DIV>
<DIV style="position:absolute;top:857;left:707"><nobr><span class="ft7">r</span></nobr></DIV>
<DIV style="position:absolute;top:862;left:713"><nobr><span class="ft14">i</span></nobr></DIV>
<DIV style="position:absolute;top:856;left:718"><nobr><span class="ft7">jg</span></nobr></DIV>
<DIV style="position:absolute;top:862;left:728"><nobr><span class="ft14">j</span></nobr></DIV>
<DIV style="position:absolute;top:856;left:737"><nobr><span class="ft7">2 U</span></nobr></DIV>
<DIV style="position:absolute;top:853;left:761"><nobr><span class="ft14">0k</span></nobr></DIV>
<DIV style="position:absolute;top:863;left:760"><nobr><span class="ft14">G</span></nobr></DIV>
<DIV style="position:absolute;top:856;left:769"><nobr><span class="ft7">Þ: ð7Þ</span></nobr></DIV>
<DIV style="position:absolute;top:885;left:434"><nobr><span class="ft16">Applying this on all the genes, we convert the ranking list to<br>a list of position p-values reflecting the significance of the<br>genes' being informative to the classification.</span></nobr></DIV>
<DIV style="position:absolute;top:938;left:452"><nobr><span class="ft7">This whole procedure for estimating the p-value of a</span></nobr></DIV>
<DIV style="position:absolute;top:956;left:434"><nobr><span class="ft12">ranking is illustrated in Fig. 1. We name this scheme the<br>r-test and call the position p-value thus calculated the r-test<br>p-value.</span></nobr></DIV>
<DIV style="position:absolute;top:1034;left:434"><nobr><span class="ft3">4</span></nobr></DIV>
<DIV style="position:absolute;top:1034;left:461"><nobr><span class="ft3">COMPENSATION FOR BIAS IN THE ESTIMATED PUGs</span></nobr></DIV>
<DIV style="position:absolute;top:1079;left:434"><nobr><span class="ft16">One important problem with the r-test scheme is the<br>selection of the PUGs. Ideally, the ranks used to select<br>PUGs and the ranks used to estimate null distribution<br>should be independent. However, this is impractical in that</span></nobr></DIV>
<DIV style="position:absolute;top:70;left:38"><nobr><span class="ft6">314</span></nobr></DIV>
<DIV style="position:absolute;top:70;left:169"><nobr><span class="ft6">IEEE/ACM TRANSACTIONS ON COMPUTATIONAL BIOLOGY AND BIOINFORMATICS,</span></nobr></DIV>
<DIV style="position:absolute;top:70;left:601"><nobr><span class="ft6">VOL. 3,</span></nobr></DIV>
<DIV style="position:absolute;top:70;left:648"><nobr><span class="ft6">NO. 3,</span></nobr></DIV>
<DIV style="position:absolute;top:70;left:689"><nobr><span class="ft6">JULY-SEPTEMBER 2006</span></nobr></DIV>
<DIV style="position:absolute;top:548;left:39"><nobr><span class="ft13">Fig. 1. The diagram showing the principle of r-test (pr-test). (a) A number (L) of new data sets is resampled from the original data set. A ranking is<br>generated for each new data set with the ranking method, resulting in a total of L rankings. (b) The genes are ordered by their average positions of<br>the L rankings. The horizontal axis is genes by this order and the vertical axis is ranking position in the resample experiments. For each gene, its<br>ranking positions in the L experiments are drawn in a box plot, with a short dash in the middle showing the median. (c) From the bottom (rightmost) of<br>the ordered gene list, k genes are selected as putative uninformative genes or PUGs. The box-plots of the ranks of the k PUGs are illustrated in this<br>enlarged image. The null distribution of ranks of uninformative genes is to be estimated from these ranks. (d) An example null distribution estimated<br>from PUGs. For each gene on the microarray, its actual ranking is compared with the null distribution to calculate the position p-value of the gene<br>being noninformative. For mr-test and tr-test, the average position in the L rankings is used in the calculation of the p*-value. In tr-test, the PUGs are<br>not selected from the bottom of the ranking, but rather from the genes with the largest t-test p-values.</span></nobr></DIV>
</DIV>
<!-- Page 4 -->
<a name="4"></a>
<DIV style="position:relative;width:918;height:1188;">
<STYLE type="text/css">
<!--
	.ft18{font-size:5px;font-family:Times;color:#000000;}
	.ft19{font-size:7px;line-height:10px;font-family:Times;color:#000000;}
-->
</STYLE>
<IMG width="918" height="1188" src="178004.png" alt="background image">
<DIV style="position:absolute;top:102;left:39"><nobr><span class="ft12">there is actually only one data set available. In our strategy,<br>the same ranks are used to estimate both PUGs and the<br>distribution of their ranks. This is an unplanned test in the<br>sense that the PUGs are defined after the ranks are observed<br>[14]. The PUGs in U</span></nobr></DIV>
<DIV style="position:absolute;top:169;left:165"><nobr><span class="ft14">0k</span></nobr></DIV>
<DIV style="position:absolute;top:179;left:164"><nobr><span class="ft14">G</span></nobr></DIV>
<DIV style="position:absolute;top:172;left:177"><nobr><span class="ft7">are not an unbiased estimate of U</span></nobr></DIV>
<DIV style="position:absolute;top:177;left:387"><nobr><span class="ft14">G</span></nobr></DIV>
<DIV style="position:absolute;top:172;left:396"><nobr><span class="ft7">. In</span></nobr></DIV>
<DIV style="position:absolute;top:189;left:39"><nobr><span class="ft12">the extreme case when k is small, uninformative genes that<br>are ranked higher are underrepresented and the ranks of<br>U</span></nobr></DIV>
<DIV style="position:absolute;top:222;left:50"><nobr><span class="ft14">0k</span></nobr></DIV>
<DIV style="position:absolute;top:232;left:49"><nobr><span class="ft14">G</span></nobr></DIV>
<DIV style="position:absolute;top:224;left:65"><nobr><span class="ft7">might represent only a tail of the ranks of U</span></nobr></DIV>
<DIV style="position:absolute;top:230;left:360"><nobr><span class="ft14">G</span></nobr></DIV>
<DIV style="position:absolute;top:224;left:374"><nobr><span class="ft7">on the</span></nobr></DIV>
<DIV style="position:absolute;top:241;left:39"><nobr><span class="ft12">right. If this happens, it will cause an overoptimistic<br>estimation of the r-test p-values and result in more genes<br>being claimed significant. Therefore, we propose two<br>modified strategies to compensate for the possible bias.</span></nobr></DIV>
<DIV style="position:absolute;top:322;left:39"><nobr><span class="ft7">4.1</span></nobr></DIV>
<DIV style="position:absolute;top:322;left:75"><nobr><span class="ft7">Modified r-Test with Average Ranks</span></nobr></DIV>
<DIV style="position:absolute;top:342;left:39"><nobr><span class="ft12">In (7), the position p-value is calculated by comparing the<br>rank r</span></nobr></DIV>
<DIV style="position:absolute;top:365;left:78"><nobr><span class="ft14">i</span></nobr></DIV>
<DIV style="position:absolute;top:360;left:86"><nobr><span class="ft7">of gene g</span></nobr></DIV>
<DIV style="position:absolute;top:365;left:143"><nobr><span class="ft14">i</span></nobr></DIV>
<DIV style="position:absolute;top:360;left:151"><nobr><span class="ft7">obtained from the whole data set with the</span></nobr></DIV>
<DIV style="position:absolute;top:377;left:39"><nobr><span class="ft12">estimated null distribution. Intuitively, when the sample size<br>is small, one single ranking based on a small sample set can<br>have a large variance, especially when all or most of the genes<br>are uninformative. We propose replacing the rank r</span></nobr></DIV>
<DIV style="position:absolute;top:435;left:358"><nobr><span class="ft14">i</span></nobr></DIV>
<DIV style="position:absolute;top:429;left:365"><nobr><span class="ft7">by the r</span></nobr></DIV>
<DIV style="position:absolute;top:435;left:412"><nobr><span class="ft14">i</span></nobr></DIV>
<DIV style="position:absolute;top:447;left:39"><nobr><span class="ft7">defined in (5), i.e., to use the average position of gene g</span></nobr></DIV>
<DIV style="position:absolute;top:452;left:374"><nobr><span class="ft14">i</span></nobr></DIV>
<DIV style="position:absolute;top:447;left:381"><nobr><span class="ft7">in the</span></nobr></DIV>
<DIV style="position:absolute;top:465;left:39"><nobr><span class="ft7">L</span></nobr></DIV>
<DIV style="position:absolute;top:464;left:54"><nobr><span class="ft7">resampling experiments as the estimate of the true rank,</span></nobr></DIV>
<DIV style="position:absolute;top:481;left:39"><nobr><span class="ft12">and to calculate the position p-value with this estimated rank<br>rather than the single observation of the rank:</span></nobr></DIV>
<DIV style="position:absolute;top:531;left:47"><nobr><span class="ft7">p</span></nobr></DIV>
<DIV style="position:absolute;top:530;left:60"><nobr><span class="ft7">ðr</span></nobr></DIV>
<DIV style="position:absolute;top:536;left:72"><nobr><span class="ft14">i</span></nobr></DIV>
<DIV style="position:absolute;top:530;left:76"><nobr><span class="ft7">Þ ¼ ^</span></nobr></DIV>
<DIV style="position:absolute;top:531;left:100"><nobr><span class="ft7">P</span></nobr></DIV>
<DIV style="position:absolute;top:530;left:117"><nobr><span class="ft7">ðr</span></nobr></DIV>
<DIV style="position:absolute;top:531;left:148"><nobr><span class="ft7">r</span></nobr></DIV>
<DIV style="position:absolute;top:536;left:154"><nobr><span class="ft14">i</span></nobr></DIV>
<DIV style="position:absolute;top:530;left:158"><nobr><span class="ft7">jg 2 U</span></nobr></DIV>
<DIV style="position:absolute;top:536;left:195"><nobr><span class="ft14">G</span></nobr></DIV>
<DIV style="position:absolute;top:530;left:204"><nobr><span class="ft7">Þ ¼ P</span></nobr></DIV>
<DIV style="position:absolute;top:536;left:237"><nobr><span class="ft14">histogram</span></nobr></DIV>
<DIV style="position:absolute;top:530;left:283"><nobr><span class="ft7">ðr</span></nobr></DIV>
<DIV style="position:absolute;top:528;left:295"><nobr><span class="ft15">l<br>j</span></nobr></DIV>
<DIV style="position:absolute;top:531;left:318"><nobr><span class="ft7">r</span></nobr></DIV>
<DIV style="position:absolute;top:536;left:325"><nobr><span class="ft14">i</span></nobr></DIV>
<DIV style="position:absolute;top:530;left:329"><nobr><span class="ft7">jg</span></nobr></DIV>
<DIV style="position:absolute;top:536;left:339"><nobr><span class="ft14">j</span></nobr></DIV>
<DIV style="position:absolute;top:530;left:348"><nobr><span class="ft7">2 U</span></nobr></DIV>
<DIV style="position:absolute;top:527;left:372"><nobr><span class="ft14">0k</span></nobr></DIV>
<DIV style="position:absolute;top:537;left:371"><nobr><span class="ft14">G</span></nobr></DIV>
<DIV style="position:absolute;top:530;left:381"><nobr><span class="ft7">Þ: ð8Þ</span></nobr></DIV>
<DIV style="position:absolute;top:559;left:39"><nobr><span class="ft12">The estimated null distribution is the distribution of single<br>ranks of putative uninformative genes, but the r</span></nobr></DIV>
<DIV style="position:absolute;top:582;left:369"><nobr><span class="ft14">i</span></nobr></DIV>
<DIV style="position:absolute;top:576;left:381"><nobr><span class="ft7">to be</span></nobr></DIV>
<DIV style="position:absolute;top:594;left:39"><nobr><span class="ft12">compared to it is the averaged rank and (8) is no longer a<br>p-value in the strict sense. Therefore, we name it p*-value<br>instead and call this modified r-test the mr-test for<br>convenience. Ideally, if a gene is informative to the<br>classification and the ranking method can consistently rank<br>the gene according this information on both the whole data<br>set and on the resampled subsets, we'll have</span></nobr></DIV>
<DIV style="position:absolute;top:727;left:168"><nobr><span class="ft7">r</span></nobr></DIV>
<DIV style="position:absolute;top:733;left:174"><nobr><span class="ft14">i</span></nobr></DIV>
<DIV style="position:absolute;top:727;left:182"><nobr><span class="ft7">¼ r</span></nobr></DIV>
<DIV style="position:absolute;top:733;left:203"><nobr><span class="ft14">i</span></nobr></DIV>
<DIV style="position:absolute;top:727;left:208"><nobr><span class="ft7">; for g</span></nobr></DIV>
<DIV style="position:absolute;top:733;left:245"><nobr><span class="ft14">i</span></nobr></DIV>
<DIV style="position:absolute;top:727;left:253"><nobr><span class="ft7">2 I</span></nobr></DIV>
<DIV style="position:absolute;top:733;left:273"><nobr><span class="ft14">G</span></nobr></DIV>
<DIV style="position:absolute;top:727;left:281"><nobr><span class="ft7">;</span></nobr></DIV>
<DIV style="position:absolute;top:727;left:398"><nobr><span class="ft7">ð9Þ</span></nobr></DIV>
<DIV style="position:absolute;top:754;left:39"><nobr><span class="ft12">in which case the p*-value will be equivalent to the original<br>r-test p-value for these genes. In practice, when the sample<br>size is small and the signal in some informative genes are<br>not so strong, we always have r</span></nobr></DIV>
<DIV style="position:absolute;top:811;left:260"><nobr><span class="ft14">i</span></nobr></DIV>
<DIV style="position:absolute;top:806;left:283"><nobr><span class="ft7">r</span></nobr></DIV>
<DIV style="position:absolute;top:811;left:289"><nobr><span class="ft14">i</span></nobr></DIV>
<DIV style="position:absolute;top:806;left:301"><nobr><span class="ft7">when r</span></nobr></DIV>
<DIV style="position:absolute;top:811;left:350"><nobr><span class="ft14">i</span></nobr></DIV>
<DIV style="position:absolute;top:806;left:361"><nobr><span class="ft7">is small;</span></nobr></DIV>
<DIV style="position:absolute;top:823;left:39"><nobr><span class="ft12">therefore, the estimated ranks move toward the right on the<br>rank distribution comparing with the single-run ranks,<br>which, as an effect, can be a compensation to the bias in the<br>estimated null distribution. (For the genes ranked in the<br>lower half of the list, the averaged rank will move leftward,<br>but these genes are not of interest to us in this study since<br>we assume only a minority of the genes can be informative.)</span></nobr></DIV>
<DIV style="position:absolute;top:956;left:39"><nobr><span class="ft7">4.2</span></nobr></DIV>
<DIV style="position:absolute;top:956;left:75"><nobr><span class="ft7">Independent Selection of PUGs</span></nobr></DIV>
<DIV style="position:absolute;top:976;left:39"><nobr><span class="ft12">The ultimate reason that may cause biased estimation of the<br>null distribution is that the PUGs in the above r-test scheme<br>are estimated from the same ranking information as that<br>being used for the calculation of the test statistics. A<br>solution is to select a group of PUGs that are an unbiased<br>sample from the U</span></nobr></DIV>
<DIV style="position:absolute;top:1069;left:173"><nobr><span class="ft14">G</span></nobr></DIV>
<DIV style="position:absolute;top:1064;left:182"><nobr><span class="ft7">. This is a big challenge because</span></nobr></DIV>
<DIV style="position:absolute;top:1081;left:39"><nobr><span class="ft7">estimating the rank position distribution of U</span></nobr></DIV>
<DIV style="position:absolute;top:1086;left:359"><nobr><span class="ft14">G</span></nobr></DIV>
<DIV style="position:absolute;top:1081;left:377"><nobr><span class="ft7">is the</span></nobr></DIV>
<DIV style="position:absolute;top:1098;left:39"><nobr><span class="ft7">question itself.</span></nobr></DIV>
<DIV style="position:absolute;top:1116;left:57"><nobr><span class="ft7">When the ranking method RM is a multivariate one such</span></nobr></DIV>
<DIV style="position:absolute;top:1133;left:39"><nobr><span class="ft7">as SVM-based methods, the ranking of the genes will not</span></nobr></DIV>
<DIV style="position:absolute;top:102;left:434"><nobr><span class="ft12">directly depend on the differences of single genes between<br>the classes. We therefore can use a univariate statistic such<br>as the t-test to select a group of nondifferentially expressed<br>genes as the PUGs since these genes will have a high<br>probability of not being informative as they are basically the<br>same in the two classes. This selection will be less correlated<br>with the ranking by RM. Applying a threshold</span></nobr></DIV>
<DIV style="position:absolute;top:209;left:768"><nobr><span class="ft7">on the</span></nobr></DIV>
<DIV style="position:absolute;top:227;left:434"><nobr><span class="ft7">t-test p-value p</span></nobr></DIV>
<DIV style="position:absolute;top:225;left:530"><nobr><span class="ft14">t</span></nobr></DIV>
<DIV style="position:absolute;top:227;left:534"><nobr><span class="ft7">, we select the PUG set U</span></nobr></DIV>
<DIV style="position:absolute;top:222;left:697"><nobr><span class="ft14">0</span></nobr></DIV>
<DIV style="position:absolute;top:234;left:696"><nobr><span class="ft14">G</span></nobr></DIV>
<DIV style="position:absolute;top:227;left:711"><nobr><span class="ft7">as:</span></nobr></DIV>
<DIV style="position:absolute;top:261;left:534"><nobr><span class="ft7">U</span></nobr></DIV>
<DIV style="position:absolute;top:257;left:545"><nobr><span class="ft14">0</span></nobr></DIV>
<DIV style="position:absolute;top:269;left:544"><nobr><span class="ft14">G</span></nobr></DIV>
<DIV style="position:absolute;top:261;left:558"><nobr><span class="ft7">¼</span></nobr></DIV>
<DIV style="position:absolute;top:256;left:559"><nobr><span class="ft14">4</span></nobr></DIV>
<DIV style="position:absolute;top:261;left:581"><nobr><span class="ft7">g</span></nobr></DIV>
<DIV style="position:absolute;top:267;left:587"><nobr><span class="ft14">j</span></nobr></DIV>
<DIV style="position:absolute;top:261;left:592"><nobr><span class="ft7">jg</span></nobr></DIV>
<DIV style="position:absolute;top:267;left:603"><nobr><span class="ft14">j</span></nobr></DIV>
<DIV style="position:absolute;top:261;left:611"><nobr><span class="ft7">2 G; p</span></nobr></DIV>
<DIV style="position:absolute;top:259;left:649"><nobr><span class="ft14">t</span></nobr></DIV>
<DIV style="position:absolute;top:261;left:653"><nobr><span class="ft7">ðg</span></nobr></DIV>
<DIV style="position:absolute;top:267;left:665"><nobr><span class="ft14">j</span></nobr></DIV>
<DIV style="position:absolute;top:261;left:670"><nobr><span class="ft7">Þ</span></nobr></DIV>
<DIV style="position:absolute;top:261;left:785"><nobr><span class="ft7">ð10Þ</span></nobr></DIV>
<DIV style="position:absolute;top:289;left:434"><nobr><span class="ft12">and estimate the null position distribution according the<br>ranking of the U</span></nobr></DIV>
<DIV style="position:absolute;top:303;left:540"><nobr><span class="ft14">0</span></nobr></DIV>
<DIV style="position:absolute;top:314;left:539"><nobr><span class="ft14">G</span></nobr></DIV>
<DIV style="position:absolute;top:307;left:553"><nobr><span class="ft7">genes by RM in the resampled data:</span></nobr></DIV>
<DIV style="position:absolute;top:335;left:511"><nobr><span class="ft7">^</span></nobr></DIV>
<DIV style="position:absolute;top:338;left:508"><nobr><span class="ft7">P</span></nobr></DIV>
<DIV style="position:absolute;top:335;left:519"><nobr><span class="ft14">t</span></nobr></DIV>
<DIV style="position:absolute;top:338;left:528"><nobr><span class="ft7">r</span></nobr></DIV>
<DIV style="position:absolute;top:338;left:535"><nobr><span class="ft7">jg 2 U</span></nobr></DIV>
<DIV style="position:absolute;top:343;left:572"><nobr><span class="ft14">G</span></nobr></DIV>
<DIV style="position:absolute;top:338;left:523"><nobr><span class="ft7">ð</span></nobr></DIV>
<DIV style="position:absolute;top:338;left:580"><nobr><span class="ft7">Þ ¼ P</span></nobr></DIV>
<DIV style="position:absolute;top:343;left:614"><nobr><span class="ft14">histogram</span></nobr></DIV>
<DIV style="position:absolute;top:338;left:659"><nobr><span class="ft7">ðr</span></nobr></DIV>
<DIV style="position:absolute;top:335;left:671"><nobr><span class="ft15">l<br>j</span></nobr></DIV>
<DIV style="position:absolute;top:338;left:676"><nobr><span class="ft7">jg</span></nobr></DIV>
<DIV style="position:absolute;top:343;left:687"><nobr><span class="ft14">j</span></nobr></DIV>
<DIV style="position:absolute;top:338;left:695"><nobr><span class="ft7">2 U</span></nobr></DIV>
<DIV style="position:absolute;top:335;left:719"><nobr><span class="ft14">0</span></nobr></DIV>
<DIV style="position:absolute;top:345;left:718"><nobr><span class="ft14">G</span></nobr></DIV>
<DIV style="position:absolute;top:338;left:727"><nobr><span class="ft7">Þ:</span></nobr></DIV>
<DIV style="position:absolute;top:338;left:785"><nobr><span class="ft7">ð11Þ</span></nobr></DIV>
<DIV style="position:absolute;top:367;left:434"><nobr><span class="ft7">The position p*-value of a gene ranked on average at r</span></nobr></DIV>
<DIV style="position:absolute;top:372;left:791"><nobr><span class="ft14">i</span></nobr></DIV>
<DIV style="position:absolute;top:367;left:800"><nobr><span class="ft7">is</span></nobr></DIV>
<DIV style="position:absolute;top:385;left:434"><nobr><span class="ft7">calculated as:</span></nobr></DIV>
<DIV style="position:absolute;top:416;left:452"><nobr><span class="ft7">^</span></nobr></DIV>
<DIV style="position:absolute;top:416;left:452"><nobr><span class="ft7">p</span></nobr></DIV>
<DIV style="position:absolute;top:416;left:464"><nobr><span class="ft7">ðr</span></nobr></DIV>
<DIV style="position:absolute;top:421;left:476"><nobr><span class="ft14">i</span></nobr></DIV>
<DIV style="position:absolute;top:416;left:480"><nobr><span class="ft7">Þ ¼ ^</span></nobr></DIV>
<DIV style="position:absolute;top:416;left:505"><nobr><span class="ft7">P</span></nobr></DIV>
<DIV style="position:absolute;top:413;left:515"><nobr><span class="ft14">t</span></nobr></DIV>
<DIV style="position:absolute;top:416;left:520"><nobr><span class="ft7">ðr</span></nobr></DIV>
<DIV style="position:absolute;top:416;left:550"><nobr><span class="ft7">r</span></nobr></DIV>
<DIV style="position:absolute;top:421;left:557"><nobr><span class="ft14">i</span></nobr></DIV>
<DIV style="position:absolute;top:416;left:561"><nobr><span class="ft7">jg 2 U</span></nobr></DIV>
<DIV style="position:absolute;top:421;left:598"><nobr><span class="ft14">G</span></nobr></DIV>
<DIV style="position:absolute;top:416;left:607"><nobr><span class="ft7">Þ ¼ P</span></nobr></DIV>
<DIV style="position:absolute;top:421;left:640"><nobr><span class="ft14">histogram</span></nobr></DIV>
<DIV style="position:absolute;top:416;left:686"><nobr><span class="ft7">ðr</span></nobr></DIV>
<DIV style="position:absolute;top:413;left:697"><nobr><span class="ft15">l<br>j</span></nobr></DIV>
<DIV style="position:absolute;top:416;left:721"><nobr><span class="ft7">r</span></nobr></DIV>
<DIV style="position:absolute;top:421;left:727"><nobr><span class="ft14">i</span></nobr></DIV>
<DIV style="position:absolute;top:416;left:732"><nobr><span class="ft7">jg</span></nobr></DIV>
<DIV style="position:absolute;top:421;left:742"><nobr><span class="ft14">j</span></nobr></DIV>
<DIV style="position:absolute;top:416;left:751"><nobr><span class="ft7">2 U</span></nobr></DIV>
<DIV style="position:absolute;top:412;left:775"><nobr><span class="ft14">0</span></nobr></DIV>
<DIV style="position:absolute;top:424;left:774"><nobr><span class="ft14">G</span></nobr></DIV>
<DIV style="position:absolute;top:416;left:783"><nobr><span class="ft7">Þ:</span></nobr></DIV>
<DIV style="position:absolute;top:439;left:785"><nobr><span class="ft7">ð12Þ</span></nobr></DIV>
<DIV style="position:absolute;top:466;left:434"><nobr><span class="ft12">For the convenience of discussion, we call this strategy the<br>tr-test and call the primary r-test defined by (7) the pr-test.<br>We view the pr-test, mr-test, and tr-test as three specific<br>methods under the general r-test scheme.</span></nobr></DIV>
<DIV style="position:absolute;top:537;left:452"><nobr><span class="ft7">It should be noted that if the ranking produced by RM is</span></nobr></DIV>
<DIV style="position:absolute;top:555;left:434"><nobr><span class="ft12">highly correlated with t-test ranking, the result of tr-test will<br>be close to that of the original pr-test. On the other hand,<br>since insignificant genes evaluated individually may not<br>necessarily be uninformative when combined with certain<br>other genes, the PUGs selected by (10) may include<br>informative genes for RM. Therefore, the estimated null<br>distribution may bias toward the left end in some situations,<br>making the results overconservative. However, in the<br>experiments described below, it is observed that the tr-test<br>results are not sensitive to changes in the p-value cut-offs<br>used for selecting PUGs (10), which is an implication that<br>the method is not very biased.</span></nobr></DIV>
<DIV style="position:absolute;top:791;left:434"><nobr><span class="ft3">5</span></nobr></DIV>
<DIV style="position:absolute;top:791;left:461"><nobr><span class="ft3">EXPERIMENTS WITH SVM ON REAL AND SIMULATED DATA</span></nobr></DIV>
<DIV style="position:absolute;top:835;left:434"><nobr><span class="ft7">5.1</span></nobr></DIV>
<DIV style="position:absolute;top:835;left:470"><nobr><span class="ft7">r-Test with SVM Gene Ranking</span></nobr></DIV>
<DIV style="position:absolute;top:855;left:434"><nobr><span class="ft12">Due to the good generalization ability of support vector<br>machines (SVM) [15], they are regarded as one of the best<br>multivariate algorithms for classifying microarray data [9],<br>[10], [16]. In the experiments for r-test in this work, we<br>adopted linear SVM as the ranking machine RM. The linear<br>SVM is trained with all genes in the data set, producing the<br>discriminate function</span></nobr></DIV>
<DIV style="position:absolute;top:991;left:563"><nobr><span class="ft7">f</span></nobr></DIV>
<DIV style="position:absolute;top:990;left:571"><nobr><span class="ft7">ðxÞ ¼ w</span></nobr></DIV>
<DIV style="position:absolute;top:991;left:637"><nobr><span class="ft7">x</span></nobr></DIV>
<DIV style="position:absolute;top:990;left:649"><nobr><span class="ft7">þ b ;</span></nobr></DIV>
<DIV style="position:absolute;top:990;left:785"><nobr><span class="ft7">ð13Þ</span></nobr></DIV>
<DIV style="position:absolute;top:1017;left:434"><nobr><span class="ft7">where w ¼ P</span></nobr></DIV>
<DIV style="position:absolute;top:1014;left:532"><nobr><span class="ft19">n<br>i</span></nobr></DIV>
<DIV style="position:absolute;top:1024;left:535"><nobr><span class="ft14">¼1</span></nobr></DIV>
<DIV style="position:absolute;top:1025;left:560"><nobr><span class="ft14">i</span></nobr></DIV>
<DIV style="position:absolute;top:1018;left:566"><nobr><span class="ft7">y</span></nobr></DIV>
<DIV style="position:absolute;top:1023;left:573"><nobr><span class="ft14">i</span></nobr></DIV>
<DIV style="position:absolute;top:1018;left:577"><nobr><span class="ft7">x</span></nobr></DIV>
<DIV style="position:absolute;top:1023;left:585"><nobr><span class="ft14">i</span></nobr></DIV>
<DIV style="position:absolute;top:1017;left:597"><nobr><span class="ft7">and</span></nobr></DIV>
<DIV style="position:absolute;top:1025;left:638"><nobr><span class="ft14">i</span></nobr></DIV>
<DIV style="position:absolute;top:1017;left:651"><nobr><span class="ft7">are the solutions of the</span></nobr></DIV>
<DIV style="position:absolute;top:1035;left:434"><nobr><span class="ft7">following quadratic programming problem:</span></nobr></DIV>
<DIV style="position:absolute;top:1076;left:491"><nobr><span class="ft7">L</span></nobr></DIV>
<DIV style="position:absolute;top:1081;left:501"><nobr><span class="ft14">p</span></nobr></DIV>
<DIV style="position:absolute;top:1075;left:511"><nobr><span class="ft7">¼ 1</span></nobr></DIV>
<DIV style="position:absolute;top:1085;left:526"><nobr><span class="ft7">2 w</span></nobr></DIV>
<DIV style="position:absolute;top:1075;left:535"><nobr><span class="ft7">k k</span></nobr></DIV>
<DIV style="position:absolute;top:1072;left:561"><nobr><span class="ft14">2</span></nobr></DIV>
<DIV style="position:absolute;top:1061;left:580"><nobr><span class="ft7">X</span></nobr></DIV>
<DIV style="position:absolute;top:1062;left:587"><nobr><span class="ft14">n</span></nobr></DIV>
<DIV style="position:absolute;top:1094;left:582"><nobr><span class="ft14">i</span></nobr></DIV>
<DIV style="position:absolute;top:1093;left:586"><nobr><span class="ft14">¼1</span></nobr></DIV>
<DIV style="position:absolute;top:1081;left:612"><nobr><span class="ft14">i</span></nobr></DIV>
<DIV style="position:absolute;top:1076;left:616"><nobr><span class="ft7">y</span></nobr></DIV>
<DIV style="position:absolute;top:1081;left:623"><nobr><span class="ft14">i</span></nobr></DIV>
<DIV style="position:absolute;top:1075;left:627"><nobr><span class="ft7">ðx</span></nobr></DIV>
<DIV style="position:absolute;top:1081;left:641"><nobr><span class="ft14">i</span></nobr></DIV>
<DIV style="position:absolute;top:1076;left:655"><nobr><span class="ft7">w</span></nobr></DIV>
<DIV style="position:absolute;top:1075;left:670"><nobr><span class="ft7">þ bÞ þ X</span></nobr></DIV>
<DIV style="position:absolute;top:1062;left:720"><nobr><span class="ft14">n</span></nobr></DIV>
<DIV style="position:absolute;top:1094;left:715"><nobr><span class="ft14">i</span></nobr></DIV>
<DIV style="position:absolute;top:1093;left:719"><nobr><span class="ft14">¼1</span></nobr></DIV>
<DIV style="position:absolute;top:1081;left:745"><nobr><span class="ft14">i</span></nobr></DIV>
<DIV style="position:absolute;top:1076;left:749"><nobr><span class="ft7">:</span></nobr></DIV>
<DIV style="position:absolute;top:1075;left:785"><nobr><span class="ft7">ð14Þ</span></nobr></DIV>
<DIV style="position:absolute;top:1115;left:434"><nobr><span class="ft12">Following [10], the contribution of each gene in the classifier<br>can be evaluated by</span></nobr></DIV>
<DIV style="position:absolute;top:70;left:38"><nobr><span class="ft6">ZHANG ET AL.: SIGNIFICANCE OF GENE RANKING FOR CLASSIFICATION OF MICROARRAY SAMPLES</span></nobr></DIV>
<DIV style="position:absolute;top:70;left:792"><nobr><span class="ft6">315</span></nobr></DIV>
</DIV>
<!-- Page 5 -->
<a name="5"></a>
<DIV style="position:relative;width:918;height:1188;">
<STYLE type="text/css">
<!--
	.ft20{font-size:12px;line-height:20px;font-family:Times;color:#000000;}
-->
</STYLE>
<IMG width="918" height="1188" src="178005.png" alt="background image">
<DIV style="position:absolute;top:107;left:126"><nobr><span class="ft7">DL</span></nobr></DIV>
<DIV style="position:absolute;top:112;left:147"><nobr><span class="ft14">p</span></nobr></DIV>
<DIV style="position:absolute;top:106;left:156"><nobr><span class="ft7">¼ ð1=2Þ @</span></nobr></DIV>
<DIV style="position:absolute;top:95;left:214"><nobr><span class="ft14">2</span></nobr></DIV>
<DIV style="position:absolute;top:97;left:220"><nobr><span class="ft7">L</span></nobr></DIV>
<DIV style="position:absolute;top:102;left:229"><nobr><span class="ft14">p</span></nobr></DIV>
<DIV style="position:absolute;top:117;left:209"><nobr><span class="ft7">@w</span></nobr></DIV>
<DIV style="position:absolute;top:115;left:227"><nobr><span class="ft14">2</span></nobr></DIV>
<DIV style="position:absolute;top:124;left:227"><nobr><span class="ft14">i</span></nobr></DIV>
<DIV style="position:absolute;top:106;left:237"><nobr><span class="ft7">ðDw</span></nobr></DIV>
<DIV style="position:absolute;top:112;left:265"><nobr><span class="ft14">i</span></nobr></DIV>
<DIV style="position:absolute;top:106;left:269"><nobr><span class="ft7">Þ</span></nobr></DIV>
<DIV style="position:absolute;top:103;left:274"><nobr><span class="ft14">2</span></nobr></DIV>
<DIV style="position:absolute;top:106;left:284"><nobr><span class="ft7">¼ ðw</span></nobr></DIV>
<DIV style="position:absolute;top:112;left:314"><nobr><span class="ft14">i</span></nobr></DIV>
<DIV style="position:absolute;top:106;left:318"><nobr><span class="ft7">Þ</span></nobr></DIV>
<DIV style="position:absolute;top:103;left:324"><nobr><span class="ft14">2</span></nobr></DIV>
<DIV style="position:absolute;top:106;left:391"><nobr><span class="ft7">ð15Þ</span></nobr></DIV>
<DIV style="position:absolute;top:143;left:39"><nobr><span class="ft7">and, thus, the genes are ranked by ðw</span></nobr></DIV>
<DIV style="position:absolute;top:149;left:290"><nobr><span class="ft14">i</span></nobr></DIV>
<DIV style="position:absolute;top:143;left:294"><nobr><span class="ft7">Þ</span></nobr></DIV>
<DIV style="position:absolute;top:140;left:300"><nobr><span class="ft14">2</span></nobr></DIV>
<DIV style="position:absolute;top:143;left:306"><nobr><span class="ft7">. There are other</span></nobr></DIV>
<DIV style="position:absolute;top:161;left:39"><nobr><span class="ft12">ways of assessing the relative contribution of the genes in a<br>SVM classifier [17], but, since the scope of this paper is not<br>to discuss the ranking method, we adopt the ranking<br>criterion given in (15) here. The ranking only reflects the<br>relative importance of the genes in the classifier, but cannot<br>reveal how important each gene is. The r-test converts the<br>ranking to position p-values (or p*-values) to evaluate the<br>significance.</span></nobr></DIV>
<DIV style="position:absolute;top:311;left:39"><nobr><span class="ft7">5.2</span></nobr></DIV>
<DIV style="position:absolute;top:311;left:75"><nobr><span class="ft7">Data Sets</span></nobr></DIV>
<DIV style="position:absolute;top:332;left:39"><nobr><span class="ft12">Experiments were done on six microarray data sets: three<br>real data sets and three simulated data sets. The leukemia<br>data set [1] contains the expression of 7,129 genes (probe<br>sets) of 72 cases, 47 of them are of the ALL class and 25 are<br>of the AML class. The colon cancer data set [18] contains<br>2,000 genes of 62 cases, among which 40 are from colon<br>cancers and 22 from normal tissues. These two data sets<br>have been widely used as benchmark sets in many<br>methodology studies. Another data set used in this study<br>is a breast cancer data set [19] containing 12,625 genes<br>(probe sets) of 85 cases. The data set is used to study the<br>classification of two subclasses of breast cancer. Forty-two<br>of the cases are of class 1 and 43 are of class 2.</span></nobr></DIV>
<DIV style="position:absolute;top:558;left:57"><nobr><span class="ft7">Simulated data sets were generated to investigate the</span></nobr></DIV>
<DIV style="position:absolute;top:576;left:39"><nobr><span class="ft12">properties of the methods in different situations. The first<br>case is for an extreme situation where none of the genes are<br>informative. The simulated data set contains 1,000 genes<br>and 100 cases. The expression values of the genes are<br>independently generated from normal distributions with<br>randomized means and variations in a given range. The<br>100 cases are generated with the same model, but are<br>assigned arbitrarily to two fake classes (50 cases in each<br>class). So, the two classes are, in fact, not separable and all<br>the genes are uninformative. We refer to this data set as the<br>"fake-class" data set in the following description.</span></nobr></DIV>
<DIV style="position:absolute;top:767;left:57"><nobr><span class="ft7">Each of the other two simulated data sets also contains</span></nobr></DIV>
<DIV style="position:absolute;top:785;left:39"><nobr><span class="ft12">1,000 genes and 100 cases of two classes (50 cases in each<br>class). In one data set (we call it "simu-1"), 700 of the genes<br>follow N(0, 1) for both classes and the 300 genes follow<br>N(0.25, 1) for class 1 and N(-0.25, 1) for class 2. In the other<br>data set (we call it "simu-2"), 700 of the genes follow N(0, 1)<br>for both classes and the 300 genes follow N(0.5, 1) for class 1<br>and N(-0.5, 1) for class 2. With these two simulated data<br>sets, we hope to mimic situations where there are weak and<br>strong classification signals in the data.</span></nobr></DIV>
<DIV style="position:absolute;top:941;left:57"><nobr><span class="ft7">All the data sets except simu-1 and simu-2 were</span></nobr></DIV>
<DIV style="position:absolute;top:959;left:39"><nobr><span class="ft12">standardized to 0-mean and standard deviation 1 first<br>across the cases and then across the genes. This is to prevent<br>possible bias in the ranking affected by the scaling. In<br>practical investigations, this step might not be needed or<br>might need to be done in some other way according to the<br>specific situation of the data and the specific ranking<br>methods to be adopted.</span></nobr></DIV>
<DIV style="position:absolute;top:1081;left:57"><nobr><span class="ft7">The six data sets used in our experiments represent</span></nobr></DIV>
<DIV style="position:absolute;top:1098;left:39"><nobr><span class="ft12">different levels of separability of the investigated classes.<br>For the leukemia data set, almost perfect classification<br>accuracy has been achieved [1], [9], [10], so it represents a</span></nobr></DIV>
<DIV style="position:absolute;top:682;left:434"><nobr><span class="ft20">relatively easy classification task. For the colon cancer data<br>set, the samples can still be well separated, but with some<br>errors [10], [18]. The two subclasses studied in the breast<br>cancer data set are hardly separable as observed in this data<br>set, but it is believed that there could be some degree of<br>separability [20], [21]. The fake-class simulation represents a<br>situation where the two classes are completely nonsepar-<br>able and the simu-1 and simu-2 simulation represents an<br>ideal situation where separation is defined on a subset of<br>the genes and the uninformative genes are i.i.d. To check<br>the classification accuracy that can be achieved on these<br>data sets, we randomly split them into independent training<br>and test sets and applied linear SVM on them. These<br>experiments were done 200 times for each data set and the<br>classification accuracy obtained at different gene selection<br>levels is summarized in Table 1. It can be seen that the<br>accuracies are consistent with the reports in the literature<br>and with the design of the simulations. (Note that the error<br>rates reported here are independent test results based on<br>only half of the samples for training, so they are larger than<br>the cross-validation errors reported elsewhere. The scope of<br>this paper is not to improve or discuss classification<br>accuracy.)</span></nobr></DIV>
<DIV style="position:absolute;top:70;left:38"><nobr><span class="ft6">316</span></nobr></DIV>
<DIV style="position:absolute;top:70;left:169"><nobr><span class="ft6">IEEE/ACM TRANSACTIONS ON COMPUTATIONAL BIOLOGY AND BIOINFORMATICS,</span></nobr></DIV>
<DIV style="position:absolute;top:70;left:601"><nobr><span class="ft6">VOL. 3,</span></nobr></DIV>
<DIV style="position:absolute;top:70;left:648"><nobr><span class="ft6">NO. 3,</span></nobr></DIV>
<DIV style="position:absolute;top:70;left:689"><nobr><span class="ft6">JULY-SEPTEMBER 2006</span></nobr></DIV>
<DIV style="position:absolute;top:107;left:595"><nobr><span class="ft4">TABLE 1</span></nobr></DIV>
<DIV style="position:absolute;top:122;left:479"><nobr><span class="ft4">Separability of the Classes of the Six Data Sets</span></nobr></DIV>
</DIV>
<!-- Page 6 -->
<a name="6"></a>
<DIV style="position:relative;width:918;height:1188;">
<STYLE type="text/css">
<!--
-->
</STYLE>
<IMG width="918" height="1188" src="178006.png" alt="background image">
<DIV style="position:absolute;top:592;left:39"><nobr><span class="ft7">5.3</span></nobr></DIV>
<DIV style="position:absolute;top:592;left:75"><nobr><span class="ft10">Number of Significant Genes According to the<br>mr-Test and tr-Test</span></nobr></DIV>
<DIV style="position:absolute;top:629;left:39"><nobr><span class="ft12">We systematically experimented with the SVM-based<br>pr-test, mr-test, and tr-test methods on the six data sets<br>and studied the number of genes claimed as significantly<br>informative with each method at various significant levels.<br>The results of the pr-test are affected by different choices of<br>the number k of selected PUGs (data not shown), indicating<br>that the pr-test can be very biased unless we know the<br>accurate number of informative genes. Therefore, we focus<br>on the mr-test and tr-test in the following discussion.</span></nobr></DIV>
<DIV style="position:absolute;top:785;left:57"><nobr><span class="ft7">Table 2 shows the number of significant genes according</span></nobr></DIV>
<DIV style="position:absolute;top:803;left:39"><nobr><span class="ft12">to the mr-test at different p*-value levels, with different<br>choices of ks on the six data sets. Comparing with the pr-test<br>results, the mr-test results are less sensitive to changes in<br>the number k. This is especially true when there are ideal<br>classification signals, as in the simu-2 data, where we can<br>see a more than 10-fold change of k causes only little<br>variance in the estimated gene numbers. With p*-value<br>levels from 0.001 to 0.1, the estimated significant gene<br>numbers are all around the correct number (300). The<br>claimed significant genes are all those true informative<br>genes in the model when the estimated genes are less than<br>300. For the situations where the number of estimated<br>informative genes is larger than 300, all the true informative<br>genes are discovered. When the data are less ideal, we see<br>that the results are stable within a smaller variation of k.<br>More experiment results with larger variations in the choice<br>of k are provided in the supplemental material, which can<br>be found on the Computer Society Digital Library at http://<br>computer.org/tcbb/archives.htm. From Table 2, it can also<br>be observed that the number of significant genes is not</span></nobr></DIV>
<DIV style="position:absolute;top:593;left:434"><nobr><span class="ft12">directly correlated with the classification accuracy. For<br>example, the breast cancer data and fake-class data both<br>look nonseparable according to the classification errors<br>(Table 1), However, for the breast cancer data, more than<br>200 genes are identified as significantly informative among<br>the 12,625 genes (</span></nobr></DIV>
<DIV style="position:absolute;top:681;left:563"><nobr><span class="ft7">1:6%</span></nobr></DIV>
<DIV style="position:absolute;top:680;left:593"><nobr><span class="ft7">) at the p*-value = 0.01 level, but,</span></nobr></DIV>
<DIV style="position:absolute;top:698;left:434"><nobr><span class="ft12">for the fake-class data, this number is only about 0.4 percent<br>of the 1,000 genes.</span></nobr></DIV>
<DIV style="position:absolute;top:732;left:452"><nobr><span class="ft7">Results of the tr-test with different t-test p-value cut-offs</span></nobr></DIV>
<DIV style="position:absolute;top:750;left:434"><nobr><span class="ft12">are shown in Table 3. It can be seen that different cut-offs<br>result in different numbers of PUGs, but the variation in<br>estimated position p*-values due to PUG number difference<br>is even smaller than in the mr-test. This implies that the<br>tr-test results are not biased by the selection of PUGs since,<br>if the PUG selection was biased, different numbers of PUGs<br>at t-test p-value cut-offs would have caused different<br>degrees of bias and the results would have varied greatly.<br>Comparing between Table 2 and Table 3, as well as the<br>results in the supplemental material, which can be found on<br>the Computer Society Digital Library at http://computer.<br>org/tcbb/archives.htm, we observe that, for the mr-test,<br>although there is a range of k for each data set in which the<br>results are not very sensitive to variations of k, this range<br>can be different with different data sets. On the other hand,<br>for the tr-test, within the same ranges of cut-off t-test p-<br>values, results on all the data sets show good consistency<br>with regard to variations in the cut-off value. This makes<br>the tr-test more applicable since users do not need to tune<br>the parameter specifically to each data set.</span></nobr></DIV>
<DIV style="position:absolute;top:1098;left:452"><nobr><span class="ft7">Comparing the number of genes selected by the tr-test</span></nobr></DIV>
<DIV style="position:absolute;top:1116;left:434"><nobr><span class="ft12">and mr-test (Table 2 and Table 3), it is obvious that the<br>tr-test is more stringent and selects much fewer genes than</span></nobr></DIV>
<DIV style="position:absolute;top:70;left:38"><nobr><span class="ft6">ZHANG ET AL.: SIGNIFICANCE OF GENE RANKING FOR CLASSIFICATION OF MICROARRAY SAMPLES</span></nobr></DIV>
<DIV style="position:absolute;top:70;left:792"><nobr><span class="ft6">317</span></nobr></DIV>
<DIV style="position:absolute;top:109;left:398"><nobr><span class="ft4">TABLE 2</span></nobr></DIV>
<DIV style="position:absolute;top:124;left:196"><nobr><span class="ft4">The Number of Genes Selected at Various r-Test p*-Value Levels with SVM</span></nobr></DIV>
</DIV>
<!-- Page 7 -->
<a name="7"></a>
<DIV style="position:relative;width:918;height:1188;">
<STYLE type="text/css">
<!--
-->
</STYLE>
<IMG width="918" height="1188" src="178007.png" alt="background image">
<DIV style="position:absolute;top:537;left:39"><nobr><span class="ft12">the mr-test on the real data sets. The differences are smaller<br>on the simulated data. Similarly to the results of the mr-test,<br>almost all the informative genes in simu-2 data can be<br>recovered at p*-value levels from 0.001 to 0.05 and there are<br>only a very few false-positive genes (e.g., the 307 genes<br>selected at p*-value = 0.05 contains all the 300 true-positive<br>genes and seven false-positive genes). This shows that the<br>SVM method is good in both sensitivity and specificity in<br>selecting the true informative genes for such ideal case, and<br>both of the two r-test methods can detect the correct<br>number of informative genes at a wide range of significance<br>levels. For simu-1 data, not all the informative genes can be<br>recovered in the experiments. This reflects the fact of the<br>large overlap of the two distributions in this weak model.<br>Many of original 300 "informative" genes are actually not<br>statistically significant in the contexts of both univariate<br>methods and multivariate methods.</span></nobr></DIV>
<DIV style="position:absolute;top:835;left:57"><nobr><span class="ft7">For the real data sets, there are no known answers for the</span></nobr></DIV>
<DIV style="position:absolute;top:852;left:39"><nobr><span class="ft12">"true" number of informative genes. The mr-test uses the tail<br>in the ranking list to estimate the null distribution for<br>assessing the significance of the genes on the top of the list,<br>therefore there is a higher possibility of the p*-values being<br>underestimated, although this has been partially compen-<br>sated by using the average rank positions. Thus, the number<br>of genes being claimed significant by mr-test might be<br>overestimated. In this sense, the tr-test scheme provides a<br>more unbiased estimation of the null distribution, which is<br>supported by the decreased sensitivity to PUG numbers.<br>With the tr-test, at the 0.05 p*-value level, we get about<br>410 significant genes from the 7,129 genes (5.75 percent) in the<br>leukemia data. On the other two real data sets, the results tend<br>to be too conservative: about 13 out of 2,000 genes<br>(0.65 percent) in the colon cancer data and 50 out of<br>12,625 genes (0.4 percent) in the breast cancer data are<br>claimed as significant at this level.</span></nobr></DIV>
<DIV style="position:absolute;top:537;left:452"><nobr><span class="ft7">It should be noted that the PUGs selected according to</span></nobr></DIV>
<DIV style="position:absolute;top:555;left:434"><nobr><span class="ft12">the t-test may contain informative genes for SVM, which<br>considers the combined effects of genes. This will cause the<br>number of genes called significant by the tr-test to be<br>underestimated. This is especially true for data sets in<br>which the major classification signal exists in the combina-<br>torial effects of genes instead of differences in single genes.<br>The correct answer may be somewhere between the two<br>estimations of the tr-test and mr-test. When the signal is<br>strong, the two estimations will be close as we see in the<br>simu-2 data. In practice, one can choose which one to use<br>according to whether the purpose is to discover more<br>possibly informative genes or to discover a more manage-<br>able set of significant genes for follow-up investigations.</span></nobr></DIV>
<DIV style="position:absolute;top:808;left:434"><nobr><span class="ft3">6</span></nobr></DIV>
<DIV style="position:absolute;top:808;left:461"><nobr><span class="ft3">DISCUSSION</span></nobr></DIV>
<DIV style="position:absolute;top:833;left:434"><nobr><span class="ft12">Statistical hypothesis testing is a fundamental framework for<br>scientific inference from observations. Unfortunately, exist-<br>ing hypothesis testing methods are not sufficient to handle<br>high-dimensional multivariate analysis problems arising<br>from current high-throughput genomic and proteomic<br>studies. Many new data mining techniques have been<br>developed both in statistics and in the machine learning<br>field. These methods are powerful in analyzing complicated<br>high-dimensional data and helped greatly in functional<br>genomics and proteomics studies. However, the analysis of<br>the statistical significance of data mining results has not been<br>paid enough attention. One reason might be that many<br>methods are rooted in techniques aimed at solving problems<br>in engineering and technological applications rather than in<br>scientific discoveries. As an example, many machine-learn-<br>ing-based gene selection and classification methods may<br>achieve very good performance in solving the specific<br>classification problems, but the results are usually of a</span></nobr></DIV>
<DIV style="position:absolute;top:70;left:38"><nobr><span class="ft6">318</span></nobr></DIV>
<DIV style="position:absolute;top:70;left:169"><nobr><span class="ft6">IEEE/ACM TRANSACTIONS ON COMPUTATIONAL BIOLOGY AND BIOINFORMATICS,</span></nobr></DIV>
<DIV style="position:absolute;top:70;left:601"><nobr><span class="ft6">VOL. 3,</span></nobr></DIV>
<DIV style="position:absolute;top:70;left:648"><nobr><span class="ft6">NO. 3,</span></nobr></DIV>
<DIV style="position:absolute;top:70;left:689"><nobr><span class="ft6">JULY-SEPTEMBER 2006</span></nobr></DIV>
<DIV style="position:absolute;top:109;left:398"><nobr><span class="ft4">TABLE 3</span></nobr></DIV>
<DIV style="position:absolute;top:124;left:161"><nobr><span class="ft4">The Number of Genes Selected at Various Position p-Value Levels by tr-Test with SVM</span></nobr></DIV>
</DIV>
<!-- Page 8 -->
<a name="8"></a>
<DIV style="position:relative;width:918;height:1188;">
<STYLE type="text/css">
<!--
-->
</STYLE>
<IMG width="918" height="1188" src="178008.png" alt="background image">
<DIV style="position:absolute;top:102;left:39"><nobr><span class="ft12">"black-box" type and judging the significance of the features<br>being used for the classification was usually not deemed<br>important. This fact compromises their further contribution<br>in helping biologists to understand the mechanisms under-<br>lying the investigated disease classification.</span></nobr></DIV>
<DIV style="position:absolute;top:190;left:57"><nobr><span class="ft7">This paper raises the problem of the significance of gene</span></nobr></DIV>
<DIV style="position:absolute;top:208;left:39"><nobr><span class="ft12">rankings in microarray classification study and proposes a<br>solution strategy called the r-test that converts the ranking<br>of genes obtained with any method to position p-values<br>(p*-values) that reflect the significance of the genes being<br>informative. The concept of this question is important and<br>the formulation and solution are challenging for several<br>reasons as addressed in the paper. First of all, the definition<br>of a gene being informative to the classification may not yet<br>be completely clear for many classification methods. Even<br>under the same criterion, there may not be a clear boundary<br>between informative and uninformative genes. A biological<br>status may be affected by several genes with different levels<br>of contribution and it may affect the expression of many<br>other genes. Differences between individuals and instru-<br>mental noises may make the genes that have no relation<br>with the studied biological process show some relevancy in<br>the limited samples. All these (and other) complexities<br>make it hard to mathematically model microarray data. We<br>propose the r-test methods based on intuitive reasoning<br>under certain assumptions about the nature of the data. As<br>shown in the experiments, the methods provide reasonable<br>solutions, but the decision by the mr-test and tr-test method<br>can be very different for some situations. Theoretically,<br>rigorous methods are still to be developed.</span></nobr></DIV>
<DIV style="position:absolute;top:635;left:57"><nobr><span class="ft7">Under the proposed r-test framework, the key issue is</span></nobr></DIV>
<DIV style="position:absolute;top:652;left:39"><nobr><span class="ft12">the choice of putative uninformative genes or PUGs. Since<br>the null distribution has to be estimated from the data<br>themselves, avoiding bias in the estimation is the most<br>challenging task. Besides the methods used by the mr-test<br>and tr-test, we have also tried several other ways to tackle<br>the problem, including selecting the PUGs according to the<br>distribution of the ranks of all genes in the resample<br>experiments, deciding the number of PUGs recursively<br>according to the rank with an EM-like strategy, selection of<br>an independent set of PUGs by fold-change, etc. Different<br>resampling strategy has also been experimented with.<br>Among these efforts, the reported mr-test and tr-test give<br>the most satisfactory results. They both perform perfectly<br>on ideal simulations. For practical cases, the mr-test has a<br>tendency to be overoptimistic by claiming more significant<br>genes and the tr-test has a tendency to be conservative by<br>approving only a small number of significant genes. Note<br>that both r-test schemes do not change the ranking itself;<br>therefore, it is the role of the classification and gene<br>selection method (the RM) to guarantee that the ranking<br>itself is reasonable for the biological investigation. The r-test<br>only helps to decide on the number of genes to be selected<br>from the list at given significance levels. Since there is<br>currently no theoretical solution to completely avoid<br>estimation bias, one can make a choice between mr-test<br>and tr-test results by balancing between the two opposite<br>trends of possible biases according to the particular<br>biological problem at hand.</span></nobr></DIV>
<DIV style="position:absolute;top:105;left:446"><nobr><span class="ft3">ACKNOWLEDGMENTS</span></nobr></DIV>
<DIV style="position:absolute;top:128;left:434"><nobr><span class="ft12">The authors would like to thank Drs. J.D. Iglehart and A.L.<br>Richardson for providing them with their microarray data<br>for the experiments. They would also like to thank the<br>editor and reviewers for their valuable suggestions that<br>contributed a lot to the work. They thank Dustin Schones<br>for helping to improve their writing. This work is supported<br>in part by NSFC projects 60275007, 60234020, and the<br>National Basic Research Program (2004CB518605) of China.<br>This work was performed while Chaolin Zhang was with<br>the MOE Key Laboratory of Bioinformatics/Department of<br>Automation, Tsinghua University, Beijing. Chaolin Zhang<br>and Xuesong Lu contributed equally in this work and<br>should be regarded as joint authors. The corresponding<br>author is Xuegong Zhang.</span></nobr></DIV>
<DIV style="position:absolute;top:401;left:446"><nobr><span class="ft3">REFERENCES</span></nobr></DIV>
<DIV style="position:absolute;top:422;left:434"><nobr><span class="ft2">[1]</span></nobr></DIV>
<DIV style="position:absolute;top:422;left:461"><nobr><span class="ft13">T.R. Golub, D.K. Slonim, P. Tamayo, C. Huard, M. Gaasenbeek,<br>J.P. Mesirov, H. Coller, M.L. Loh, J.R. Downing, M.A. Caligiuri,<br>C.D. Bloomfield, and E.S. Lander, "Molecular Classification of<br>Cancer: Class Discovery and Class Prediction by Gene Expression<br>Monitoring," Science, vol. 286, no. 5439, pp. 531-537, 1999.</span></nobr></DIV>
<DIV style="position:absolute;top:489;left:434"><nobr><span class="ft2">[2]</span></nobr></DIV>
<DIV style="position:absolute;top:489;left:461"><nobr><span class="ft13">C.-A. Tsai, Y.-J. Chen, and J.J. Chen, "Testing for Differentially<br>Expressed Genes with Microarray Data," Nuclear Acids Research,<br>vol. 31, no. 9, p. e52, 2003.</span></nobr></DIV>
<DIV style="position:absolute;top:529;left:434"><nobr><span class="ft2">[3]</span></nobr></DIV>
<DIV style="position:absolute;top:529;left:461"><nobr><span class="ft13">M.S. Pepe, G. Longton, G.L. Anderson, and M. Schummer,<br>"Selecting Differentially Expressed Genes from Microarray Ex-<br>periments," Biometrics, vol. 59, no. 1, pp. 133-142, 2003.</span></nobr></DIV>
<DIV style="position:absolute;top:570;left:434"><nobr><span class="ft2">[4]</span></nobr></DIV>
<DIV style="position:absolute;top:570;left:461"><nobr><span class="ft13">P. Broberg, "Statistical Methods for Ranking Differentially<br>Expressed Genes," Genome Biology, vol. 4, no. 6, p. R41, 2003.</span></nobr></DIV>
<DIV style="position:absolute;top:597;left:434"><nobr><span class="ft2">[5]</span></nobr></DIV>
<DIV style="position:absolute;top:597;left:461"><nobr><span class="ft13">W. Pan, "A Comparative Review of Statistical Methods for<br>Discovering Differentially Expressed Genes in Replicated Micro-<br>array Experiments," Bioinformatics, vol. 18, no. 4, pp. 546-554, 2002.</span></nobr></DIV>
<DIV style="position:absolute;top:637;left:434"><nobr><span class="ft2">[6]</span></nobr></DIV>
<DIV style="position:absolute;top:637;left:461"><nobr><span class="ft13">S. Ramaswamy, K.N. Ross, E.S. Lander, and T.R. Golub, "A<br>Molecular Signature of Metastasis in Primary Solid Tumors,"<br>Nature Genetics, vol. 33, no. 1, pp. 49-54, 2003.</span></nobr></DIV>
<DIV style="position:absolute;top:677;left:434"><nobr><span class="ft2">[7]</span></nobr></DIV>
<DIV style="position:absolute;top:677;left:461"><nobr><span class="ft13">L.J. van 't Veer, H. Dai, M.J. van de Vijver, Y.D. He, A.A.M. Hart,<br>M. Mao, H.L. Peterse, K. van der Kooy, M.J. Marton, A.T.<br>Witteveen, G.J. Schreiber, R.M. Kerkhoven, C. Roberts, P.S.<br>Linsley, R. Bernards, and S.H. Friend, "Gene Expression Profiling<br>Predicts Clinical Outcome of Breast Cancer," Nature, vol. 415,<br>no. 6871, pp. 530-536, 2002.</span></nobr></DIV>
<DIV style="position:absolute;top:758;left:434"><nobr><span class="ft2">[8]</span></nobr></DIV>
<DIV style="position:absolute;top:758;left:461"><nobr><span class="ft13">M. Xiong, X. Fang, and J. Zhao, "Biomarker Identification by<br>Feature Wrappers," Genome Research, vol. 11, no. 11, pp. 1878-1887,<br>2001.</span></nobr></DIV>
<DIV style="position:absolute;top:798;left:434"><nobr><span class="ft2">[9]</span></nobr></DIV>
<DIV style="position:absolute;top:798;left:461"><nobr><span class="ft13">X. Zhang and H. Ke, "ALL/AML Cancer Classification by Gene<br>Expression Data Using SVM and CSVM Approach," Proc. Conf.<br>Genome Informatics, pp. 237-239, 2000.</span></nobr></DIV>
<DIV style="position:absolute;top:839;left:434"><nobr><span class="ft2">[10]</span></nobr></DIV>
<DIV style="position:absolute;top:839;left:461"><nobr><span class="ft13">I. Guyon, J. Weston, S. Barnhill, and V. Vapnik, "Gene Selection<br>for Cancer Classification Using Support Vector Machines,"<br>Machine Learning, vol. 46, pp. 389-422, 2002.</span></nobr></DIV>
<DIV style="position:absolute;top:879;left:434"><nobr><span class="ft2">[11]</span></nobr></DIV>
<DIV style="position:absolute;top:879;left:461"><nobr><span class="ft13">C. Furlanello, M. Serafini, S. Merler, and G. Jurman, "Entropy-<br>Based Gene Ranking without Selection Bias for the Predictive<br>Classification of Microarray Data," BMC Bioinformatics, vol. 4,<br>no. 1, p. 54, 2003.</span></nobr></DIV>
<DIV style="position:absolute;top:933;left:434"><nobr><span class="ft2">[12]</span></nobr></DIV>
<DIV style="position:absolute;top:933;left:461"><nobr><span class="ft13">H. Yu, J. Yang, W. Wang, and J. Han, "Discovering Compact and<br>Highly Discriminative Features or Feature Combinations of Drug<br>Activities Using Support Vector Machines," Proc. 2003 IEEE<br>Bioinformatics Conf. (CSB '03), 2003.</span></nobr></DIV>
<DIV style="position:absolute;top:987;left:434"><nobr><span class="ft2">[13]</span></nobr></DIV>
<DIV style="position:absolute;top:987;left:461"><nobr><span class="ft13">J.D. Storey and R. Tibshirani, "Statistical Significance for Genome<br>Wide Studies," Proc. Nat'l Academy of Science USA, vol. 100, no. 16,<br>pp. 9440-9445, 2003.</span></nobr></DIV>
<DIV style="position:absolute;top:1027;left:434"><nobr><span class="ft2">[14]</span></nobr></DIV>
<DIV style="position:absolute;top:1027;left:461"><nobr><span class="ft2">R.R. Sokal and F.J. Rohlf, Biometry. San Francisco: Freeman, 1995.</span></nobr></DIV>
<DIV style="position:absolute;top:1041;left:434"><nobr><span class="ft2">[15]</span></nobr></DIV>
<DIV style="position:absolute;top:1040;left:461"><nobr><span class="ft13">V. Vapnik, The Nature of Statistical Learning Theory. Springer-<br>Verlag, 1995.</span></nobr></DIV>
<DIV style="position:absolute;top:1067;left:434"><nobr><span class="ft2">[16]</span></nobr></DIV>
<DIV style="position:absolute;top:1067;left:461"><nobr><span class="ft13">S. Ramaswamy, P. Tamayo, R. Rifkin, S. Mukherjee, C.-H. Yeang,<br>M. Angelo, C. Ladd, M. Reich, E. Latulippe, J.P. Mesirov, T.<br>Poggio, W. Gerald, M. Loda, E.S. Lander, and T.R. Golub,<br>"Multiclass Cancer Diagnosis Using Tumor Gene Expression<br>Signatures," Proc. Nat'l Academy of Sciences, vol. 98, no. 26,<br>pp. 15149-15154, 2001.</span></nobr></DIV>
<DIV style="position:absolute;top:70;left:38"><nobr><span class="ft6">ZHANG ET AL.: SIGNIFICANCE OF GENE RANKING FOR CLASSIFICATION OF MICROARRAY SAMPLES</span></nobr></DIV>
<DIV style="position:absolute;top:70;left:792"><nobr><span class="ft6">319</span></nobr></DIV>
</DIV>
<!-- Page 9 -->
<a name="9"></a>
<DIV style="position:relative;width:918;height:1188;">
<STYLE type="text/css">
<!--
-->
</STYLE>
<IMG width="918" height="1188" src="178009.png" alt="background image">
<DIV style="position:absolute;top:100;left:39"><nobr><span class="ft2">[17]</span></nobr></DIV>
<DIV style="position:absolute;top:100;left:66"><nobr><span class="ft13">X. Zhang and W.H. Wong, "Recursive Sample Classification and<br>Gene Selection Based on SVM: Method and Software Descrip-<br>tion," technical report, Dept. of Biostatistics, Harvard School of<br>Public Health, 2001.</span></nobr></DIV>
<DIV style="position:absolute;top:154;left:39"><nobr><span class="ft2">[18]</span></nobr></DIV>
<DIV style="position:absolute;top:154;left:66"><nobr><span class="ft13">U. Alon, N. Barkai, D.A. Notterman, K. Gish, S. Ybarra, D. Mack,<br>and A.J. Levine, "Broad Patterns of Gene Expression Revealed by<br>Clustering Analysis of Tumor and Normal Colon Tissues Probed<br>by Oligonucleotide Arrays," Proc. Nat'l Academy of Sciences, vol. 96,<br>no. 12, pp. 6745-6750, 1999.</span></nobr></DIV>
<DIV style="position:absolute;top:221;left:39"><nobr><span class="ft2">[19]</span></nobr></DIV>
<DIV style="position:absolute;top:221;left:66"><nobr><span class="ft13">Z.C. Wang, M. Lin, L.-J. Wei, C. Li, A. Miron, G. Lodeiro, L.<br>Harris, S. Ramaswamy, D.M. Tanenbaum, M. Meyerson, J.D.<br>Iglehart, and A. Richardson, "Loss of Heterozygosity and Its<br>Correlation with Expression Profiles in Subclasses of Invasive<br>Breast Cancers," Cancer Research, vol. 64, no. 1, pp. 64-71, 2004.</span></nobr></DIV>
<DIV style="position:absolute;top:288;left:39"><nobr><span class="ft2">[20]</span></nobr></DIV>
<DIV style="position:absolute;top:288;left:66"><nobr><span class="ft13">E. Huang, S.H. Cheng, H. Dressman, J. Pittman, M.H. Tsou, C.F.<br>Horng, A. Bild, E.S. Iversen, M. Liao, and C.M. Chen, "Gene<br>Expression Predictors of Breast Cancer Outcomes," The Lancet,<br>vol. 361, no. 9369, pp. 1590-1596, 2003.</span></nobr></DIV>
<DIV style="position:absolute;top:342;left:39"><nobr><span class="ft2">[21]</span></nobr></DIV>
<DIV style="position:absolute;top:342;left:66"><nobr><span class="ft13">M. West, C. Blanchette, H. Dressman, E. Huang, S. Ishida, R.<br>Spang, H. Zuzan, J.A. Olson Jr., J.R. Marks, and J.R. Nevins,<br>"Predicting the Clinical Status of Human Breast Cancer by Using<br>Gene Expression Profiles," Proc. Nat'l Academy of Sciences, vol. 98,<br>no. 20, pp. 11462-11467, 2001.</span></nobr></DIV>
<DIV style="position:absolute;top:424;left:163"><nobr><span class="ft13">Chaolin Zhang received the BE degree from<br>the Department of Automation at Tsinghua<br>University, Beijing, China, in 2002. From 2002<br>to 2004, he worked as a graduate student on<br>machine learning applications in microarray data<br>analysis and literature mining at the MOE Key<br>Laboratory of Bioinformatics, Tsinghua Univer-<br>sity. He is now a PhD student at Cold Spring<br>Harbor Laboratory and the Department of<br>Biomedical Engineering, the State University of</span></nobr></DIV>
<DIV style="position:absolute;top:559;left:39"><nobr><span class="ft2">New York at Stony Brook.</span></nobr></DIV>
<DIV style="position:absolute;top:100;left:557"><nobr><span class="ft13">Xuesong Lu received the BE degree from the<br>Department of Automation, Tsinghua University,<br>Beijing, China, in 2001. He is currently a PhD<br>candidate in the Department of Automation and<br>the MOE Key Laboratory of Bioinformatics at<br>Tsinghua University, Beijing, China. His re-<br>search interests include microarray data mining,<br>gene network modeling, and literature mining.</span></nobr></DIV>
<DIV style="position:absolute;top:271;left:557"><nobr><span class="ft13">Xuegong Zhang received the PhD degree in<br>pattern recognition and intelligent systems from<br>Tsinghua University, Beijing, China, in 1994. He<br>is currently a professor in the Department of<br>Automation and the MOE Key Laboratory of<br>Bioinformatics at Tsinghua University. His re-<br>search interests include machine learning and<br>pattern recognition, bioinformatics, computa-<br>tional genomics, and systems biology.</span></nobr></DIV>
<DIV style="position:absolute;top:475;left:434"><nobr><span class="ft2">.</span></nobr></DIV>
<DIV style="position:absolute;top:474;left:446"><nobr><span class="ft2">For more information on this or any other computing topic,</span></nobr></DIV>
<DIV style="position:absolute;top:488;left:434"><nobr><span class="ft2">please visit our Digital Library at www.computer.org/publications/dlib.</span></nobr></DIV>
<DIV style="position:absolute;top:70;left:38"><nobr><span class="ft6">320</span></nobr></DIV>
<DIV style="position:absolute;top:70;left:169"><nobr><span class="ft6">IEEE/ACM TRANSACTIONS ON COMPUTATIONAL BIOLOGY AND BIOINFORMATICS,</span></nobr></DIV>
<DIV style="position:absolute;top:70;left:601"><nobr><span class="ft6">VOL. 3,</span></nobr></DIV>
<DIV style="position:absolute;top:70;left:648"><nobr><span class="ft6">NO. 3,</span></nobr></DIV>
<DIV style="position:absolute;top:70;left:689"><nobr><span class="ft6">JULY-SEPTEMBER 2006</span></nobr></DIV>
</DIV>
</BODY>
</HTML>
